{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "b846a376",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('NR08 phage', attributes={'MajorTopicYN': 'N'}), StringElement('Xanthomonas oryzae pv. oryzae', attributes={'MajorTopicYN': 'N'}), StringElement('bacterial leaf blight', attributes={'MajorTopicYN': 'N'}), StringElement('bacteriophages', attributes={'MajorTopicYN': 'N'}), StringElement('biocontrol', attributes={'MajorTopicYN': 'N'}), StringElement('rice', attributes={'MajorTopicYN': 'N'}), StringElement('whole-genome sequencing', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('37007514', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '04', 'Day': '04'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '03', 'Day': '16'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('1084025', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2023.1084025', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1664-302X', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '14', 'PubDate': {'Year': '2023'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Frontiers in microbiology', 'ISOAbbreviation': 'Front Microbiol'}, 'ArticleTitle': 'Isolation of bacteriophages infecting <i>Xanthomonas oryzae</i> pv. <i>oryzae</i> and genomic characterization of novel phage vB_XooS_NR08 for biocontrol of bacterial leaf blight of rice.', 'Pagination': {'StartPage': '1084025', 'MedlinePgn': '1084025'}, 'Abstract': {'AbstractText': ['Bacterial leaf blight (BLB) disease of rice caused by <i>Xanthomonas oryzae</i> pv<i>. oryzae</i> (<i>Xoo</i>) is one of the most destructive diseases worldwide in rice-growing regions. The Ineffectiveness of chemicals in disease management has increased the interest in phage therapy. In this study, we isolated 19 bacteriophages, infecting <i>Xoo</i>, from a rice field, which belonged to phage families <i>Siphoviridae, Myoviridae</i>, and <i>Podoviridae</i> on the basis of electron microscopy. Among 19 phages, Phage vB_XooS_NR08, a member of the <i>Siphoviridae</i> family, expressed antibacterial activity against all <i>Xoo</i> strains tested and did not lyse <i>X. campestris</i> and other unrelated bacterial hosts. Phage NR08 showed more than 80% viability at a temperature range of 4°C-40°C, pH range of 5-9, and direct exposure to sunlight for 2 h, whereas UV light and chemical agents were highly detrimental. In a one-step growth curve, NR08 has a 40-min latent period, followed by a 30-min burst period with a burst size of 250 particle/bacterium. The genome of NR08 is double-stranded DNA, linear having a size of 98,812 bp with a G + C content of 52.9%. Annotation of the whole-genome sequence indicated that NR08 encodes 142 putative open reading frames (ORFs), including one ORF for tRNA, namely, trna1-GlnTTG. Comparative genome analysis of NR08 showed that it shares maximum similarity with <i>Pseudomonas</i> phage PaMx42 (40% query coverage, 95.39% identity, and acc. Length 43,225) and <i>Xanthomonas</i> phage Samson (40% query coverage, 96.68% identity, and acc. Length 43,314). The average alignment percentage (AP) of NR08 with other Xoophages was only 0.32 to 1.25 since the genome of NR08 (98.8 kb) is almost double of most of the previously reported Xoophages (43-47 kb), thus indicating NR08 a novel Xoophage. In <i>in vitro</i> bacterial challenge assay, NR08 showed bacteriostasis up to 24 h and a 99.95% reduction in bacterial growth in 48 h. In rice pot efficacy trials, single-dose treatment of NR08 showed a significant reduction in disease up to 90.23% and 79.27% on 7 and 21 dpi, respectively. However, treatment using 2% skim milk-supplemented phage preparation was significantly less effective as compared to the neat phage preparation. In summary, this study characterized a novel Xoophage having the potential as a biocontrol agent in the mitigation of BLB in rice.'], 'CopyrightInformation': 'Copyright © 2023 Jain, Kumar, Jain, Kaushal and Ghosh.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'ICAR-National Institute of Biotic Stress Management, Raipur, Chhattisgarh, India.'}], 'Identifier': [], 'LastName': 'Jain', 'ForeName': 'Lata', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'ICAR-National Institute of Biotic Stress Management, Raipur, Chhattisgarh, India.'}], 'Identifier': [], 'LastName': 'Kumar', 'ForeName': 'Vinay', 'Initials': 'V'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'ICAR-National Institute of Biotic Stress Management, Raipur, Chhattisgarh, India.'}], 'Identifier': [], 'LastName': 'Jain', 'ForeName': 'Sanjay Kumar', 'Initials': 'SK'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'ICAR-National Institute of Biotic Stress Management, Raipur, Chhattisgarh, India.'}], 'Identifier': [], 'LastName': 'Kaushal', 'ForeName': 'Pankaj', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'ICAR-National Institute of Biotic Stress Management, Raipur, Chhattisgarh, India.'}], 'Identifier': [], 'LastName': 'Ghosh', 'ForeName': 'Probir Kumar', 'Initials': 'PK'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Switzerland', 'MedlineTA': 'Front Microbiol', 'NlmUniqueID': '101548977', 'ISSNLinking': '1664-302X'}, 'CoiStatement': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['Bacterial leaf blight (BLB) disease of rice caused by <i>Xanthomonas oryzae</i> pv<i>. oryzae</i> (<i>Xoo</i>) is one of the most destructive diseases worldwide in rice-growing regions. The Ineffectiveness of chemicals in disease management has increased the interest in phage therapy. In this study, we isolated 19 bacteriophages, infecting <i>Xoo</i>, from a rice field, which belonged to phage families <i>Siphoviridae, Myoviridae</i>, and <i>Podoviridae</i> on the basis of electron microscopy. Among 19 phages, Phage vB_XooS_NR08, a member of the <i>Siphoviridae</i> family, expressed antibacterial activity against all <i>Xoo</i> strains tested and did not lyse <i>X. campestris</i> and other unrelated bacterial hosts. Phage NR08 showed more than 80% viability at a temperature range of 4°C-40°C, pH range of 5-9, and direct exposure to sunlight for 2 h, whereas UV light and chemical agents were highly detrimental. In a one-step growth curve, NR08 has a 40-min latent period, followed by a 30-min burst period with a burst size of 250 particle/bacterium. The genome of NR08 is double-stranded DNA, linear having a size of 98,812 bp with a G + C content of 52.9%. Annotation of the whole-genome sequence indicated that NR08 encodes 142 putative open reading frames (ORFs), including one ORF for tRNA, namely, trna1-GlnTTG. Comparative genome analysis of NR08 showed that it shares maximum similarity with <i>Pseudomonas</i> phage PaMx42 (40% query coverage, 95.39% identity, and acc. Length 43,225) and <i>Xanthomonas</i> phage Samson (40% query coverage, 96.68% identity, and acc. Length 43,314). The average alignment percentage (AP) of NR08 with other Xoophages was only 0.32 to 1.25 since the genome of NR08 (98.8 kb) is almost double of most of the previously reported Xoophages (43-47 kb), thus indicating NR08 a novel Xoophage. In <i>in vitro</i> bacterial challenge assay, NR08 showed bacteriostasis up to 24 h and a 99.95% reduction in bacterial growth in 48 h. In rice pot efficacy trials, single-dose treatment of NR08 showed a significant reduction in disease up to 90.23% and 79.27% on 7 and 21 dpi, respectively. However, treatment using 2% skim milk-supplemented phage preparation was significantly less effective as compared to the neat phage preparation. In summary, this study characterized a novel Xoophage having the potential as a biocontrol agent in the mitigation of BLB in rice.']\n",
      "[StringElement('1084025', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2023.1084025', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Isolation of bacteriophages infecting <i>Xanthomonas oryzae</i> pv. <i>oryzae</i> and genomic characterization of novel phage vB_XooS_NR08 for biocontrol of bacterial leaf blight of rice.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Collinsella urealyticum sp. nov.', attributes={'MajorTopicYN': 'N'}), StringElement('Gut microbiota', attributes={'MajorTopicYN': 'N'}), StringElement('Swine faeces', attributes={'MajorTopicYN': 'N'}), StringElement('Taxonomy', attributes={'MajorTopicYN': 'N'}), StringElement('Urease', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('37004685', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '04', 'Day': '04'}, 'DateRevised': {'Year': '2023', 'Month': '04', 'Day': '04'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '04', 'Day': '01'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1007/s00203-023-03510-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1432-072X', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '205', 'Issue': '4', 'PubDate': {'Year': '2023', 'Month': 'Apr', 'Day': '01'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Archives of microbiology', 'ISOAbbreviation': 'Arch Microbiol'}, 'ArticleTitle': 'Collinsella urealyticum sp. nov., a urease-positive bacterial strain isolated from swine faeces.', 'Pagination': {'StartPage': '156', 'MedlinePgn': '156'}, 'Abstract': {'AbstractText': [\"A novel actinobacterial strain, designated AGMB00827<sup>T</sup>, was isolated from swine faeces. Strain AGMB00827<sup>T</sup> was obligately anaerobic, Gram-stain-positive, non-motile, non-spore-forming and rod-shaped bacterium. Comparative analyses based on the 16S rRNA gene and whole genome sequence revealed that strain AGMB00827<sup>T</sup> was affiliated to the genus Collinsella, and was most closely related to Collinsella vaginalis Marseille-P2666<sup>T</sup> (=\\u2009KCTC 25056<sup>T</sup>). Biochemical analysis showed strain AGMB00827<sup>T</sup> was negative for catalase and oxidase. Interestingly, strain AGMB00827<sup>T</sup> possessed urease activity, which was determined by traditional methods (API test and Christensen's urea medium), unlike related strains. Furthermore, the major cellular fatty acids (>\\u200910%) of the isolate were C<sub>18:1</sub> ω9c, C<sub>16:0</sub>, C<sub>16:0</sub> DMA and C<sub>18:2</sub> ω9,12c DMA. Based on the whole genome sequence analysis, the DNA G\\u2009+\\u2009C content of strain AGMB00827<sup>T</sup> was 52.3%, and the genome size and numbers of rRNA and tRNA genes were 1,945,251\\xa0bp, 3 and 46, respectively. The average nucleotide identity and digital DNA-DNA hybridization values between strain AGMB00827<sup>T</sup> and C. vaginalis KCTC 25056<sup>\\xa0T</sup> were 71.0 and 23.2%, respectively. Additionally, the genome analysis revealed that strain AGMB00827<sup>T</sup> possesses urease gene cluster including ureABC and ureDEFG while the related strains do not have those genes, which is consistent with the urease activity. On the basis of polyphasic taxonomic approach, strain AGMB00827<sup>T</sup> represents a novel species within the genus Collinsella, for which the name Collinsella urealyticum sp. nov. is proposed. The type strain is AGMB00827<sup>T</sup> (=\\u2009KCTC 25287<sup>T</sup>\\u2009=\\u2009GDMCC 1.2724<sup>T</sup>).\"], 'CopyrightInformation': '© 2023. The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, part of Springer Nature.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Oh', 'ForeName': 'Byeong Seob', 'Initials': 'BS'}, attributes={'ValidYN': 'Y', 'EqualContrib': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Ryu', 'ForeName': 'Seoung Woo', 'Initials': 'SW'}, attributes={'ValidYN': 'Y', 'EqualContrib': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Yu', 'ForeName': 'Seung Yeob', 'Initials': 'SY'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Bak', 'ForeName': 'Jeong Eun', 'Initials': 'JE'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Choi', 'ForeName': 'Won Jung', 'Initials': 'WJ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kim', 'ForeName': 'Ji-Sun', 'Initials': 'JS'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}, {'Identifier': [], 'Affiliation': 'University of Science and Technology (UST), Daejeon, 34113, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Lee', 'ForeName': 'Jung-Sook', 'Initials': 'JS'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Park', 'ForeName': 'Seung-Hwan', 'Initials': 'SH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kang', 'ForeName': 'Se Won', 'Initials': 'SW'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}, {'Identifier': [], 'Affiliation': 'University of Science and Technology (UST), Daejeon, 34113, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Lee', 'ForeName': 'Jiyoung', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea.'}, {'Identifier': [], 'Affiliation': 'University of Science and Technology (UST), Daejeon, 34113, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Lee', 'ForeName': 'Mi-Kyung', 'Initials': 'MK'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon, 34141, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Jung', 'ForeName': 'Won Yong', 'Initials': 'WY'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Institute of Animal Science, Cheonan, 31000, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kim', 'ForeName': 'Jo Eun', 'Initials': 'JE'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Institute of Animal Science, Cheonan, 31000, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Choi', 'ForeName': 'Yo Han', 'Initials': 'YH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Animal Resources Science, Dankook University, Cheonan, 31116, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kim', 'ForeName': 'Hyeun Bum', 'Initials': 'HB'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Advanced Radiation Technology Institute, Korea Atomic Energy Research Institute, Jeongeup, 56212, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kim', 'ForeName': 'Jae-Kyung', 'Initials': 'JK'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Food and Animal Biotechnology, Department of Agricultural Biotechnology, Center for Food and Bioconvergence, Seoul National University, Seoul, 08826, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Lee', 'ForeName': 'Ju-Hoon', 'Initials': 'JH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Korean Collection for Type Cultures, Biological Resource Center, Korea Research Institute of Bioscience and Biotechnology, Jeongeup, 56212, Republic of Korea. juhuck@kribb.re.kr.'}, {'Identifier': [], 'Affiliation': 'University of Science and Technology (UST), Daejeon, 34113, Republic of Korea. juhuck@kribb.re.kr.'}], 'Identifier': [], 'LastName': 'Lee', 'ForeName': 'Ju Huck', 'Initials': 'JH'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic'}), 'MedlineJournalInfo': {'Country': 'Germany', 'MedlineTA': 'Arch Microbiol', 'NlmUniqueID': '0410427', 'ISSNLinking': '0302-8933'}, 'ChemicalList': [{'RegistryNumber': 'EC 3.5.1.5', 'NameOfSubstance': StringElement('Urease', attributes={'UI': 'D014510'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('DNA, Bacterial', attributes={'UI': 'D004269'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Fatty Acids', attributes={'UI': 'D005227'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Phospholipids', attributes={'UI': 'D010743'})}], 'MeshHeadingList': [{'QualifierName': [], 'DescriptorName': StringElement('Animals', attributes={'UI': 'D000818', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Swine', attributes={'UI': 'D013552', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Phylogeny', attributes={'UI': 'D010802', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Urease', attributes={'UI': 'D014510', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('DNA, Bacterial', attributes={'UI': 'D004269', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('analysis', attributes={'UI': 'Q000032', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Fatty Acids', attributes={'UI': 'D005227', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Feces', attributes={'UI': 'D005243', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacterial Typing Techniques', attributes={'UI': 'D015373', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Sequence Analysis, DNA', attributes={'UI': 'D017422', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('analysis', attributes={'UI': 'Q000032', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Phospholipids', attributes={'UI': 'D010743', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "[\"A novel actinobacterial strain, designated AGMB00827<sup>T</sup>, was isolated from swine faeces. Strain AGMB00827<sup>T</sup> was obligately anaerobic, Gram-stain-positive, non-motile, non-spore-forming and rod-shaped bacterium. Comparative analyses based on the 16S rRNA gene and whole genome sequence revealed that strain AGMB00827<sup>T</sup> was affiliated to the genus Collinsella, and was most closely related to Collinsella vaginalis Marseille-P2666<sup>T</sup> (=\\u2009KCTC 25056<sup>T</sup>). Biochemical analysis showed strain AGMB00827<sup>T</sup> was negative for catalase and oxidase. Interestingly, strain AGMB00827<sup>T</sup> possessed urease activity, which was determined by traditional methods (API test and Christensen's urea medium), unlike related strains. Furthermore, the major cellular fatty acids (>\\u200910%) of the isolate were C<sub>18:1</sub> ω9c, C<sub>16:0</sub>, C<sub>16:0</sub> DMA and C<sub>18:2</sub> ω9,12c DMA. Based on the whole genome sequence analysis, the DNA G\\u2009+\\u2009C content of strain AGMB00827<sup>T</sup> was 52.3%, and the genome size and numbers of rRNA and tRNA genes were 1,945,251\\xa0bp, 3 and 46, respectively. The average nucleotide identity and digital DNA-DNA hybridization values between strain AGMB00827<sup>T</sup> and C. vaginalis KCTC 25056<sup>\\xa0T</sup> were 71.0 and 23.2%, respectively. Additionally, the genome analysis revealed that strain AGMB00827<sup>T</sup> possesses urease gene cluster including ureABC and ureDEFG while the related strains do not have those genes, which is consistent with the urease activity. On the basis of polyphasic taxonomic approach, strain AGMB00827<sup>T</sup> represents a novel species within the genus Collinsella, for which the name Collinsella urealyticum sp. nov. is proposed. The type strain is AGMB00827<sup>T</sup> (=\\u2009KCTC 25287<sup>T</sup>\\u2009=\\u2009GDMCC 1.2724<sup>T</sup>).\"]\n",
      "[StringElement('10.1007/s00203-023-03510-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Collinsella urealyticum sp. nov., a urease-positive bacterial strain isolated from swine faeces.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Bacillus subtilis TY-1', attributes={'MajorTopicYN': 'N'}), StringElement('Biological control', attributes={'MajorTopicYN': 'N'}), StringElement('Deep sea', attributes={'MajorTopicYN': 'N'}), StringElement('Genome sequencing', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36894216', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '03', 'Day': '13'}, 'DateRevised': {'Year': '2023', 'Month': '03', 'Day': '13'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '02', 'Day': '24'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1016/j.margen.2023.101018', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S1874-7787(23)00010-7', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1876-7478', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '68', 'PubDate': {'Year': '2023', 'Month': 'Apr'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Marine genomics', 'ISOAbbreviation': 'Mar Genomics'}, 'ArticleTitle': 'Complete genome analysis of Bacillus subtilis TY-1 reveals its biocontrol potential against tobacco bacterial wilt.', 'Pagination': {'StartPage': '101018', 'MedlinePgn': '101018'}, 'Abstract': {'AbstractText': ['Bacillus subtilis TY-1 was isolated from 2000\\xa0m-deep sea sediments of the Western Pacific Ocean, which was found to exhibit strong antagonistic activity against tobacco bacterial wilt caused by Ralstonia solanacearum. Here, we present the annotated complete genomic sequence of the strain Bacillus subtilis TY-1. The genome consists of a 4,030,869-bp circular chromosome with a G\\xa0+\\xa0C content of 43.88%, 86 tRNAs, and 30 rRNAs. Genomic analysis identified a large number of gene clusters involved in the biosynthesis of antibacterial metabolites, including lipopeptides(surfactin, bacillibactin, and fengycin) and polyketides(bacillaene). Meanwhile, numerous genes encoding carbohydrate-active enzymes and secreted proteins were found in TY-1. These findings suggest that Bacillus subtilis TY-1 appears to be a potential biocontrol agent against tobacco bacterial wilt in agricultural fields.'], 'CopyrightInformation': 'Copyright © 2023 Elsevier B.V. All rights reserved.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Advanced Manufacturing, Fuzhou University, Jinjiang 362200, Fujian, China; Key Laboratory of Marine Genetic Resources, Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, Fujian, China.'}], 'Identifier': [], 'LastName': 'Tian', 'ForeName': 'Yin', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Technology Center, China Tobacco Fujian Industrial Co., Ltd., Xiamen 361021, Fujian, China.'}], 'Identifier': [], 'LastName': 'Ji', 'ForeName': 'Shunhua', 'Initials': 'S'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Technology Center, China Tobacco Fujian Industrial Co., Ltd., Xiamen 361021, Fujian, China.'}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Enren', 'Initials': 'E'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Technology Center, China Tobacco Fujian Industrial Co., Ltd., Xiamen 361021, Fujian, China.'}], 'Identifier': [], 'LastName': 'Chen', 'ForeName': 'Yiqiang', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Key Laboratory of Marine Genetic Resources, Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, Fujian, China.'}], 'Identifier': [], 'LastName': 'Xu', 'ForeName': 'Guangxin', 'Initials': 'G'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Key Laboratory of Marine Genetic Resources, Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, Fujian, China.'}], 'Identifier': [], 'LastName': 'Chen', 'ForeName': 'Xi', 'Initials': 'X'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Technology Center, China Tobacco Fujian Industrial Co., Ltd., Xiamen 361021, Fujian, China. Electronic address: fjq10393@fjtic.cn.'}], 'Identifier': [], 'LastName': 'Fan', 'ForeName': 'Jianqiang', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Advanced Manufacturing, Fuzhou University, Jinjiang 362200, Fujian, China; Key Laboratory of Marine Genetic Resources, Third Institute of Oceanography, Ministry of Natural Resources, Xiamen 361005, Fujian, China. Electronic address: tangxixiang@tio.org.cn.'}], 'Identifier': [], 'LastName': 'Tang', 'ForeName': 'Xixiang', 'Initials': 'X'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'Netherlands', 'MedlineTA': 'Mar Genomics', 'NlmUniqueID': '101475200', 'ISSNLinking': '1874-7787'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('Anti-Bacterial Agents', attributes={'UI': 'D000900'})}], 'MeshHeadingList': [{'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'}), StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Bacillus subtilis', attributes={'UI': 'D001412', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'}), StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Tobacco', attributes={'UI': 'D014026', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Genome, Bacterial', attributes={'UI': 'D016680', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Anti-Bacterial Agents', attributes={'UI': 'D000900', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Genomics', attributes={'UI': 'D023281', 'MajorTopicYN': 'N'})}], 'CoiStatement': 'Declaration of Competing Interest The authors declare no conflict of interest.'}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['Bacillus subtilis TY-1 was isolated from 2000\\xa0m-deep sea sediments of the Western Pacific Ocean, which was found to exhibit strong antagonistic activity against tobacco bacterial wilt caused by Ralstonia solanacearum. Here, we present the annotated complete genomic sequence of the strain Bacillus subtilis TY-1. The genome consists of a 4,030,869-bp circular chromosome with a G\\xa0+\\xa0C content of 43.88%, 86 tRNAs, and 30 rRNAs. Genomic analysis identified a large number of gene clusters involved in the biosynthesis of antibacterial metabolites, including lipopeptides(surfactin, bacillibactin, and fengycin) and polyketides(bacillaene). Meanwhile, numerous genes encoding carbohydrate-active enzymes and secreted proteins were found in TY-1. These findings suggest that Bacillus subtilis TY-1 appears to be a potential biocontrol agent against tobacco bacterial wilt in agricultural fields.']\n",
      "[StringElement('10.1016/j.margen.2023.101018', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S1874-7787(23)00010-7', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})]\n",
      "Complete genome analysis of Bacillus subtilis TY-1 reveals its biocontrol potential against tobacco bacterial wilt.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Lactic acid bacteria', attributes={'MajorTopicYN': 'N'}), StringElement('Lactobacillus', attributes={'MajorTopicYN': 'N'}), StringElement('Makgeolli', attributes={'MajorTopicYN': 'N'}), StringElement('Metabolite', attributes={'MajorTopicYN': 'N'}), StringElement('Microbial community', attributes={'MajorTopicYN': 'N'}), StringElement('Rice wine', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36845507', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '28'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '23'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('100552', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1016/j.fochx.2022.100552', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('2590-1575', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '17', 'PubDate': {'Year': '2023', 'Month': 'Mar', 'Day': '30'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Food chemistry: X', 'ISOAbbreviation': 'Food Chem X'}, 'ArticleTitle': 'Investigation of lactic acid bacterial profiles in commercial rice wine and their effect on metabolites during low-temperature storage.', 'Pagination': {'StartPage': '100552', 'MedlinePgn': '100552'}, 'Abstract': {'AbstractText': ['Makgeolli, the traditional Korean rice wine, is generally considered to contain lactic acid bacteria (LAB) despite its bacterial inoculation-free brewing process. The existence of LAB in makgeolli often presents inconsistent trends in microbial profiles and cell numbers. Therefore, to establish LAB-related insights, 94 commercial non-pasteurized products were collected and microbial communities and metabolites were analyzed using 16S rRNA amplicon sequencing and GC-MS, respectively. All samples contained various LAB genera and species, with an average viable cell number of 5.61 log CFU/mL. Overall, 10 LAB genera and 25 LAB species were detected; the most abundant and frequent LAB genus was <i>Lactobacillus</i>. There was no significant change in the LAB composition profile or lactic acid content during low-temperature storage, indicating the presence of LAB did not significantly affect the quality of makgeolli under low-temperature storage conditions. Overall, this study contributes to understand the microbial profile and role of LAB in makgeolli.'], 'CopyrightInformation': '© 2022 The Author(s).'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Biotechnology, College of Life Sciences and Biotechnology, Korea University, Seoul 02841, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Cha', 'ForeName': 'Jeongmin', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'AccuGene Inc., Incheon 21999, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Cho', 'ForeName': 'Kwang-Moon', 'Initials': 'KM'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'AccuGene Inc., Incheon 21999, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kwon', 'ForeName': 'Sun Jae', 'Initials': 'SJ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Biotechnology, College of Life Sciences and Biotechnology, Korea University, Seoul 02841, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Park', 'ForeName': 'Seong-Eun', 'Initials': 'SE'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Biotechnology, College of Life Sciences and Biotechnology, Korea University, Seoul 02841, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Kim', 'ForeName': 'Eun-Ju', 'Initials': 'EJ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Sonlab Inc., Seoul 02841, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Seo', 'ForeName': 'Seung-Ho', 'Initials': 'SH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Biotechnology, College of Life Sciences and Biotechnology, Korea University, Seoul 02841, Republic of Korea.'}], 'Identifier': [], 'LastName': 'Son', 'ForeName': 'Hong-Seok', 'Initials': 'HS'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Netherlands', 'MedlineTA': 'Food Chem X', 'NlmUniqueID': '101751436', 'ISSNLinking': '2590-1575'}, 'CoiStatement': 'The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "['Makgeolli, the traditional Korean rice wine, is generally considered to contain lactic acid bacteria (LAB) despite its bacterial inoculation-free brewing process. The existence of LAB in makgeolli often presents inconsistent trends in microbial profiles and cell numbers. Therefore, to establish LAB-related insights, 94 commercial non-pasteurized products were collected and microbial communities and metabolites were analyzed using 16S rRNA amplicon sequencing and GC-MS, respectively. All samples contained various LAB genera and species, with an average viable cell number of 5.61 log CFU/mL. Overall, 10 LAB genera and 25 LAB species were detected; the most abundant and frequent LAB genus was <i>Lactobacillus</i>. There was no significant change in the LAB composition profile or lactic acid content during low-temperature storage, indicating the presence of LAB did not significantly affect the quality of makgeolli under low-temperature storage conditions. Overall, this study contributes to understand the microbial profile and role of LAB in makgeolli.']\n",
      "[StringElement('100552', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1016/j.fochx.2022.100552', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Investigation of lactic acid bacterial profiles in commercial rice wine and their effect on metabolites during low-temperature storage.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('ammonia oxidizing microorga-nisms', attributes={'MajorTopicYN': 'N'}), StringElement('litter nitrogen', attributes={'MajorTopicYN': 'N'}), StringElement('microbial functional guilds', attributes={'MajorTopicYN': 'N'}), StringElement('soil bacterial community', attributes={'MajorTopicYN': 'N'}), StringElement('tree species', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [DictElement({'AbstractText': ['为揭示亚热带人工林常见造林树种对森林土壤微生物群落的影响,本研究选取马尾松、米老排、枫香、冬青、火力楠、麻栎和光皮桦7个树种为研究对象,采用16S rRNA高通量测序和实时荧光定量PCR技术,探究不同树种土壤细菌的多样性、群落构成以及微生物功能群基因丰度。结果表明: 变形菌门、酸杆菌门和放线菌门是亚热带造林树种的优势细菌门,不同树种细菌多样性和丰富度指数无显著差异。冗余分析表明,土壤容重、土壤C/N、凋落物氮和凋落物C/N是影响土壤细菌组成的主要环境因子。不同造林树种土壤中氨氧化古菌、氨氧化细菌和完全氨氧化菌<i>amoA</i>基因丰度均具有显著差异。完全氨氧化菌在数量上占据优势地位,但只有氨氧化古菌<i>amoA</i>基因丰度与土壤硝态氮呈显著正相关关系,表明氨氧化古菌在亚热带酸性森林土壤自养硝化作用中可能发挥主要作用。相关分析表明,凋落物氮是不同树种影响氨氧化微生物丰度变化的关键驱动因子。本研究表明,土壤微生物功能群对树种的响应比细菌群落结构更加敏感,未来应从微生物功能群角度深入探究不同造林树种对森林生态系统功能的影响机制。.']}, attributes={'Type': 'Publisher', 'Language': 'chi'})], 'PMID': StringElement('36799399', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '02', 'Day': '22'}, 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '22'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [], 'ELocationID': [StringElement('10.13287/j.1001-9332.202212.034', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1001-9332', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '34', 'Issue': '1', 'PubDate': {'Year': '2023', 'Month': 'Jan'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Ying yong sheng tai xue bao = The journal of applied ecology', 'ISOAbbreviation': 'Ying Yong Sheng Tai Xue Bao'}, 'ArticleTitle': 'Effects of common afforestation tree species on soil bacterial community and microbial functional guilds in subtropical forests.', 'Pagination': {'StartPage': '235', 'EndPage': '241', 'MedlinePgn': '235-241'}, 'Abstract': {'AbstractText': ['To understand the effects of common afforestation tree species on soil microbial community in subtropical forests, seven different tree species were selected as the research object, including <i>Pinus massoniana</i>, <i>Mytilaria laosensis</i>, <i>Liquidambar formosana</i>, <i>Ilex chinensis</i>, <i>Michelia macclurei</i>, <i>Quercus acutissima</i> and <i>Betula luminifera</i>. Based on 16S rRNA high-throughput sequencing and real-time quantitative PCR techniques, we explored the effects of different tree species on soil bacterial community composition, diversity and microbial functional guilds. The results showed that Acidobacteria, Proteobacteria, and Actinobacteria were the dominant bacterial phyla, and that there was no significant difference in bacterial diversity or richness index among different tree species. Results of redundancy analysis suggested that soil bulk density, soil C/N, litter nitrogen content, and litter C/N were the predominant factors determining soil bacterial community composition. The afforestation tree species had significant effects on functional gene abundances of ammonia oxidizing archaea, ammonia oxidizing bacteria and complete ammonia oxidation. Comammox were dominant in abundance. Ammonia oxidizing archaea <i>amoA</i> gene was the only type whose abundance showed significant correlation with soil nitrate content, suggesting that ammonia oxidizing archaea could play a dominant role in the autotrophic nitrification in the acidic subtropical forest soils. The afforestation tree species had significant effects on functional gene abundances of ammonia oxidizing microorganisms. Results of correlation analysis showed that litter nitrogen content was the driving factor for the abundance of ammonia oxidizing microorganisms. Our study provided strong evidence that the responses of soil microbial functional guilds to tree species were more sensitive than bacterial community composition. Future studies should explore the mechanisms of tree plantations on forest ecosystem functioning from the perspective of microbial functional guilds.']}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Shao', 'ForeName': 'Ya-Jun', 'Initials': 'YJ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Li-Yan', 'Initials': 'LY'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Tan', 'ForeName': 'Yun-Yan', 'Initials': 'YY'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Leng', 'ForeName': 'Peng', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Jian-Qing', 'Initials': 'JQ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Xu', 'ForeName': 'Jing-Hua', 'Initials': 'JH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Geographical Science, Fujian Normal University, Fuzhou 350007, China.'}, {'Identifier': [], 'Affiliation': 'Cultivation Base of State Key Laboratory of Humid Subtropical Mountain Ecology, Fuzhou 350007, China.'}], 'Identifier': [], 'LastName': 'Shi', 'ForeName': 'Xiu-Zhen', 'Initials': 'XZ'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})], 'VernacularTitle': '亚热带常见造林树种对土壤细菌及微生物功能群的影响.'}, attributes={'PubModel': 'Print'}), 'MedlineJournalInfo': {'Country': 'China', 'MedlineTA': 'Ying Yong Sheng Tai Xue Bao', 'NlmUniqueID': '9425159', 'ISSNLinking': '1001-9332'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('Soil', attributes={'UI': 'D012987'})}, {'RegistryNumber': '7664-41-7', 'NameOfSubstance': StringElement('Ammonia', attributes={'UI': 'D000641'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': 'N762921K75', 'NameOfSubstance': StringElement('Nitrogen', attributes={'UI': 'D009584'})}], 'MeshHeadingList': [{'QualifierName': [], 'DescriptorName': StringElement('Trees', attributes={'UI': 'D014197', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Soil', attributes={'UI': 'D012987', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Ammonia', attributes={'UI': 'D000641', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Oxidation-Reduction', attributes={'UI': 'D010084', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Archaea', attributes={'UI': 'D001105', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Forests', attributes={'UI': 'D065928', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Nitrification', attributes={'UI': 'D058465', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Microbiota', attributes={'UI': 'D064307', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Nitrogen', attributes={'UI': 'D009584', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Soil Microbiology', attributes={'UI': 'D012988', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Phylogeny', attributes={'UI': 'D010802', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['To understand the effects of common afforestation tree species on soil microbial community in subtropical forests, seven different tree species were selected as the research object, including <i>Pinus massoniana</i>, <i>Mytilaria laosensis</i>, <i>Liquidambar formosana</i>, <i>Ilex chinensis</i>, <i>Michelia macclurei</i>, <i>Quercus acutissima</i> and <i>Betula luminifera</i>. Based on 16S rRNA high-throughput sequencing and real-time quantitative PCR techniques, we explored the effects of different tree species on soil bacterial community composition, diversity and microbial functional guilds. The results showed that Acidobacteria, Proteobacteria, and Actinobacteria were the dominant bacterial phyla, and that there was no significant difference in bacterial diversity or richness index among different tree species. Results of redundancy analysis suggested that soil bulk density, soil C/N, litter nitrogen content, and litter C/N were the predominant factors determining soil bacterial community composition. The afforestation tree species had significant effects on functional gene abundances of ammonia oxidizing archaea, ammonia oxidizing bacteria and complete ammonia oxidation. Comammox were dominant in abundance. Ammonia oxidizing archaea <i>amoA</i> gene was the only type whose abundance showed significant correlation with soil nitrate content, suggesting that ammonia oxidizing archaea could play a dominant role in the autotrophic nitrification in the acidic subtropical forest soils. The afforestation tree species had significant effects on functional gene abundances of ammonia oxidizing microorganisms. Results of correlation analysis showed that litter nitrogen content was the driving factor for the abundance of ammonia oxidizing microorganisms. Our study provided strong evidence that the responses of soil microbial functional guilds to tree species were more sensitive than bacterial community composition. Future studies should explore the mechanisms of tree plantations on forest ecosystem functioning from the perspective of microbial functional guilds.']\n",
      "[StringElement('10.13287/j.1001-9332.202212.034', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Effects of common afforestation tree species on soil bacterial community and microbial functional guilds in subtropical forests.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36780625', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '02', 'Day': '15'}, 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '15'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [], 'ELocationID': [], 'Journal': {'ISSN': StringElement('1512-0112', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Issue': '333', 'PubDate': {'Year': '2022', 'Month': 'Dec'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Georgian medical news', 'ISOAbbreviation': 'Georgian Med News'}, 'ArticleTitle': 'BACTERIAL CONTAMINATION AND METHODS OF DECONTAMINATION OF BASES COMPLETE REMOVABLE PROSTHESES DURING THE APPLICATION OF ADHESIVE MATERIALS.', 'Pagination': {'StartPage': '61', 'EndPage': '66', 'MedlinePgn': '61-66'}, 'Abstract': {'AbstractText': ['It turned out that the permanent microbiota of surfaces of acrylic bases of complete removable plastic dentures (CRPD) characterized by several features: the presence of aerobic (7) and anaerobic (10) species. Qualitative and quantitative compositions of microbiocenosis in orthopedic treatment stages have been changed. The revealed patterns require further study in the context of the improvement of decolonization of acrylic bases in the stages of orthopedic treatment by CRPD. Deactivation of acrylic bases of CRPD significantly affects the species composition of microbiota, reduces the quantitative parameters of bacterial contamination. The purpose of our work was determination of bacterial contamination of acrylic bases of CRPD in stages of their clinical exploitation and study of changes in the quantitative composition of the acrylic bases microbiota of CRPD under different decontamination modes. Smears from the inner surface of the acrylic base were stained with Gram and microscopic and seeded on blood agar, Endo agar, Chistovich, Saburo. Species belonging to the micro-organisms were identified by the LAHEMA test system and were determined in colonies forming units (CFUs). These data indicate accumulation under the acrylic base in the course of clinical operation of certain types of microbiota, which requires improvement of the methods of their decontamination. We studied the change in the quantitative and specific composition of the microbiota (bacterial contamination) of acrylic bases CRPD under different modes (\"A\", \"B\"). Patients in group \"A\" used the Sideex solution - a two-component system consisting of a liquid component (glutaraldehyde solution) and a powdery activator, mixed before use to obtain a working activated solution. The liquid component is a clear, colorless solution with a specific odor, which is a 2.2-2.7% aqueous solution of glutaraldehyde, which is an active ingredient, pH=3.0 - 4.5. Powdery activator is a pale-yellow powder containing alkaline components, a corrosion inhibitor, and a dye. The activated working solution is a fluorescent green solution with a specific odor and containing 2.2-2.7% glutaraldehyde; pH=8.2-9.2. (Great Britain). Preparation of the activated solution: a powder-activator is added to the container containing the liquid component (avoiding losses). Cleaning is carried out by the method of full immersion of the prosthesis in solution, with a thickness of the drug over it not less than 1 cm. The dentures are soused in solution for 15 minutes, then thorough washing in the same solution for 1-3 minutes. Sideex activated solution is used for sterilization and disinfection of metal, glass, polymeric (plastic, rubber, etc.) medical products. The dentures of patients from group \"B\" are decontaminated with 0.2% solution of chlorhexidine bigluconate. The dentures were placed in the solution overnight for 14 days, changing the solution every two days. The decontamination of acrylic bases of CRPD should be considered mode \"B\" as more effective because of the significant decrease in the species composition of the microbiota (the qualitative indicator of reduction of microbial contamination was 72.0%). It is substantiated that decontamination of acrylic bases of CRPD with mode \"A\" has a small effect on the species composition of the microbiota, although it significantly (p<0.05) reduces quantitative indicators of microbial contamination. Regarding \"B\" mode, the species composition of aerobes from 31 lgCFU/ml to 8.7 lgCFU/ml and anaerobes from 42.7 lgCFU/ml to 14.6 lgCFU/ml was considered more effective.']}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'Movchan', 'ForeName': 'O', 'Initials': 'O'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'Yanishen', 'ForeName': 'I', 'Initials': 'I'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'Diudina', 'ForeName': 'I', 'Initials': 'I'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'Tomilin', 'ForeName': 'V', 'Initials': 'V'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'German', 'ForeName': 'S', 'Initials': 'S'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Kharkiv National Medical University, Ukraine.'}], 'Identifier': [], 'LastName': 'Pereshyvailova', 'ForeName': 'I', 'Initials': 'I'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print'}), 'MedlineJournalInfo': {'Country': 'Georgia (Republic)', 'MedlineTA': 'Georgian Med News', 'NlmUniqueID': '101218222', 'ISSNLinking': '1512-0112'}, 'ChemicalList': [{'RegistryNumber': 'T3C89M417N', 'NameOfSubstance': StringElement('Glutaral', attributes={'UI': 'D005976'})}, {'RegistryNumber': '9002-18-0', 'NameOfSubstance': StringElement('Agar', attributes={'UI': 'D000362'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Powders', attributes={'UI': 'D011208'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Plastics', attributes={'UI': 'D010969'})}], 'MeshHeadingList': [{'QualifierName': [], 'DescriptorName': StringElement('Humans', attributes={'UI': 'D006801', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Glutaral', attributes={'UI': 'D005976', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Agar', attributes={'UI': 'D000362', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Decontamination', attributes={'UI': 'D003666', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Powders', attributes={'UI': 'D011208', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Denture, Complete', attributes={'UI': 'D003824', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Plastics', attributes={'UI': 'D010969', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Denture Bases', attributes={'UI': 'D003819', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['It turned out that the permanent microbiota of surfaces of acrylic bases of complete removable plastic dentures (CRPD) characterized by several features: the presence of aerobic (7) and anaerobic (10) species. Qualitative and quantitative compositions of microbiocenosis in orthopedic treatment stages have been changed. The revealed patterns require further study in the context of the improvement of decolonization of acrylic bases in the stages of orthopedic treatment by CRPD. Deactivation of acrylic bases of CRPD significantly affects the species composition of microbiota, reduces the quantitative parameters of bacterial contamination. The purpose of our work was determination of bacterial contamination of acrylic bases of CRPD in stages of their clinical exploitation and study of changes in the quantitative composition of the acrylic bases microbiota of CRPD under different decontamination modes. Smears from the inner surface of the acrylic base were stained with Gram and microscopic and seeded on blood agar, Endo agar, Chistovich, Saburo. Species belonging to the micro-organisms were identified by the LAHEMA test system and were determined in colonies forming units (CFUs). These data indicate accumulation under the acrylic base in the course of clinical operation of certain types of microbiota, which requires improvement of the methods of their decontamination. We studied the change in the quantitative and specific composition of the microbiota (bacterial contamination) of acrylic bases CRPD under different modes (\"A\", \"B\"). Patients in group \"A\" used the Sideex solution - a two-component system consisting of a liquid component (glutaraldehyde solution) and a powdery activator, mixed before use to obtain a working activated solution. The liquid component is a clear, colorless solution with a specific odor, which is a 2.2-2.7% aqueous solution of glutaraldehyde, which is an active ingredient, pH=3.0 - 4.5. Powdery activator is a pale-yellow powder containing alkaline components, a corrosion inhibitor, and a dye. The activated working solution is a fluorescent green solution with a specific odor and containing 2.2-2.7% glutaraldehyde; pH=8.2-9.2. (Great Britain). Preparation of the activated solution: a powder-activator is added to the container containing the liquid component (avoiding losses). Cleaning is carried out by the method of full immersion of the prosthesis in solution, with a thickness of the drug over it not less than 1 cm. The dentures are soused in solution for 15 minutes, then thorough washing in the same solution for 1-3 minutes. Sideex activated solution is used for sterilization and disinfection of metal, glass, polymeric (plastic, rubber, etc.) medical products. The dentures of patients from group \"B\" are decontaminated with 0.2% solution of chlorhexidine bigluconate. The dentures were placed in the solution overnight for 14 days, changing the solution every two days. The decontamination of acrylic bases of CRPD should be considered mode \"B\" as more effective because of the significant decrease in the species composition of the microbiota (the qualitative indicator of reduction of microbial contamination was 72.0%). It is substantiated that decontamination of acrylic bases of CRPD with mode \"A\" has a small effect on the species composition of the microbiota, although it significantly (p<0.05) reduces quantitative indicators of microbial contamination. Regarding \"B\" mode, the species composition of aerobes from 31 lgCFU/ml to 8.7 lgCFU/ml and anaerobes from 42.7 lgCFU/ml to 14.6 lgCFU/ml was considered more effective.']\n",
      "\n",
      "BACTERIAL CONTAMINATION AND METHODS OF DECONTAMINATION OF BASES COMPLETE REMOVABLE PROSTHESES DURING THE APPLICATION OF ADHESIVE MATERIALS.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Verticillium wilt', attributes={'MajorTopicYN': 'N'}), StringElement('amplicon sequencing', attributes={'MajorTopicYN': 'N'}), StringElement('continuous cropping obstacle', attributes={'MajorTopicYN': 'N'}), StringElement('cotton', attributes={'MajorTopicYN': 'N'}), StringElement('soil microbe', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36778850', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '14'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '01', 'Day': '26'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('1125564', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2023.1125564', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1664-302X', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '14', 'PubDate': {'Year': '2023'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Frontiers in microbiology', 'ISOAbbreviation': 'Front Microbiol'}, 'ArticleTitle': 'Soil bacterial community response to continuous cropping of cotton.', 'Pagination': {'StartPage': '1125564', 'MedlinePgn': '1125564'}, 'Abstract': {'AbstractText': [StringElement('Long-term continuous cropping may result in the outbreak and proliferation of soil-borne diseases, as well as reduction in annual crop production. Overcoming the obstacles of continuous cropping is critical for the long-term growth of modern agriculture. Soil microbes are essential for plant health, but the consequences of continuous cropping on soil microbiome are still poorly understood.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('This study analyzed changes in soil bacterial community composition of Aksu (AKS) and Shihezi (SHZ) in Xinjiang Province during 1-20\\u2009 years of continuous cropping by 16S amplicon sequencing. The results showed that the incidence of cotton Verticillium wilt rose with the number of cropping years. The bacterial alpha diversity in the AKS soil grew as the number of continuous cropping years increased, however it declined in the SHZ soil.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The results of beta diversity analysis showed that there were significant differences in soil bacterial communities between different continuous cropping years and between different soils. The results of community composition changes at the level of main phyla and genus showed that the relative abundance of Actinobacteria, Bacteroidetes and <i>Streptomyces</i> decreased with the increase of continuous cropping years in the AKS and the SHZ soils. In addition, Actinobacteria, Propionibacteriales, and Nocardioidaceae were significantly enriched during the early stages of continuous cropping. Network analysis showed that long-term (≥8\\u2009 years) continuous cropping interfered with the complexity of soil bacterial co-occurrence networks and reduced collaboration between OTUs.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('These findings suggested that continuous cropping and soil origin jointly affected the diversity and structural of bacterial communities, and the loss of Nocardioidaceae and <i>Streptomyces</i> in Actinobacteria might be one of the reasons of continuous cropping obstacles.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})], 'CopyrightInformation': 'Copyright © 2023 Ma, Li, Yang, Feng, Feng, Zhang, Zhao, Zhou, Zhu and Wei.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Engineering Research Center of Cotton Biology Breeding and Industrial Technology, Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Ma', 'ForeName': 'Zheng', 'Initials': 'Z'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Peng', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Yang', 'ForeName': 'Chuanzhen', 'Initials': 'C'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Engineering Research Center of Cotton Biology Breeding and Industrial Technology, Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Feng', 'ForeName': 'Zili', 'Initials': 'Z'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Feng', 'ForeName': 'Hongjie', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Engineering Research Center of Cotton Biology Breeding and Industrial Technology, Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Yalin', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Engineering Research Center of Cotton Biology Breeding and Industrial Technology, Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Zhao', 'ForeName': 'Lihong', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}], 'Identifier': [], 'LastName': 'Zhou', 'ForeName': 'Jinglong', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Zhu', 'ForeName': 'Heqin', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'National Engineering Research Center of Cotton Biology Breeding and Industrial Technology, Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Institute of Cotton Research of Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Wei', 'ForeName': 'Feng', 'Initials': 'F'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Switzerland', 'MedlineTA': 'Front Microbiol', 'NlmUniqueID': '101548977', 'ISSNLinking': '1664-302X'}, 'CoiStatement': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "[StringElement('Long-term continuous cropping may result in the outbreak and proliferation of soil-borne diseases, as well as reduction in annual crop production. Overcoming the obstacles of continuous cropping is critical for the long-term growth of modern agriculture. Soil microbes are essential for plant health, but the consequences of continuous cropping on soil microbiome are still poorly understood.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('This study analyzed changes in soil bacterial community composition of Aksu (AKS) and Shihezi (SHZ) in Xinjiang Province during 1-20\\u2009 years of continuous cropping by 16S amplicon sequencing. The results showed that the incidence of cotton Verticillium wilt rose with the number of cropping years. The bacterial alpha diversity in the AKS soil grew as the number of continuous cropping years increased, however it declined in the SHZ soil.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The results of beta diversity analysis showed that there were significant differences in soil bacterial communities between different continuous cropping years and between different soils. The results of community composition changes at the level of main phyla and genus showed that the relative abundance of Actinobacteria, Bacteroidetes and <i>Streptomyces</i> decreased with the increase of continuous cropping years in the AKS and the SHZ soils. In addition, Actinobacteria, Propionibacteriales, and Nocardioidaceae were significantly enriched during the early stages of continuous cropping. Network analysis showed that long-term (≥8\\u2009 years) continuous cropping interfered with the complexity of soil bacterial co-occurrence networks and reduced collaboration between OTUs.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('These findings suggested that continuous cropping and soil origin jointly affected the diversity and structural of bacterial communities, and the loss of Nocardioidaceae and <i>Streptomyces</i> in Actinobacteria might be one of the reasons of continuous cropping obstacles.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})]\n",
      "[StringElement('1125564', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2023.1125564', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Soil bacterial community response to continuous cropping of cotton.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36692291', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '18'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '01', 'Day': '24'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('e01237-22', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1128/mra.01237-22', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('2576-098X', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '12', 'Issue': '2', 'PubDate': {'Year': '2023', 'Month': 'Feb', 'Day': '16'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Microbiology resource announcements', 'ISOAbbreviation': 'Microbiol Resour Announc'}, 'ArticleTitle': 'Genome Assembly of a Putative Plant Growth-Stimulating Bacterial Sweet Pepper Fruit Isolate, Enterobacter hormaechei SRU4.4.', 'Pagination': {'StartPage': 'e0123722', 'MedlinePgn': 'e0123722'}, 'Abstract': {'AbstractText': ['Here, we report the draft genome sequence of Enterobacter hormaechei SRU4.4. This bacterium (genome size\\u2009=\\u20094,440,516\\u2009bp; coding sequences\\u2009=\\u20094,100; G+C content\\u2009=\\u200956%) encodes for genes attributed to plant growth promotion (PGP).']}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Gastro-Intestinal Microbiology and Biotechnology Unit, Agricultural Research Council-Animal Production, Irene, Pretoria, South Africa.'}], 'Identifier': [StringElement('0000-0003-4763-8952', attributes={'Source': 'ORCID'})], 'LastName': 'Mamphogoro', 'ForeName': 'Tshifhiwa Paris', 'Initials': 'TP'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Plant Production Sciences and Technologies, University of Zimbabwe, Mount Pleasant, Harare, Zimbabwe.'}], 'Identifier': [StringElement('0000-0001-5878-2162', attributes={'Source': 'ORCID'})], 'LastName': 'Kamutando', 'ForeName': 'Casper Nyaradzai', 'Initials': 'CN'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Crop Sciences, Tshwane University of Technology, Pretoria, South Africa.'}], 'Identifier': [StringElement('0000-0003-0690-6455', attributes={'Source': 'ORCID'})], 'LastName': 'Maboko', 'ForeName': 'Martin Makgose', 'Initials': 'MM'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Food Security and Safety Focus Area, Faculty of Natural and Agricultural Sciences, North-West University, Mmabatho, South Africa.'}], 'Identifier': [StringElement('0000-0003-4344-1909', attributes={'Source': 'ORCID'})], 'LastName': 'Babalola', 'ForeName': 'Olubukola Oluranti', 'Initials': 'OO'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Research Unit for Environmental Sciences and Management, North-West University, Potchefstroom, South Africa.'}], 'Identifier': [StringElement('0000-0001-6664-6574', attributes={'Source': 'ORCID'})], 'LastName': 'Aiyegoro', 'ForeName': 'Olayinka Ayobami', 'Initials': 'OA'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'United States', 'MedlineTA': 'Microbiol Resour Announc', 'NlmUniqueID': '101728794', 'ISSNLinking': '2576-098X'}, 'CoiStatement': 'The authors declare no conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "['Here, we report the draft genome sequence of Enterobacter hormaechei SRU4.4. This bacterium (genome size\\u2009=\\u20094,440,516\\u2009bp; coding sequences\\u2009=\\u20094,100; G+C content\\u2009=\\u200956%) encodes for genes attributed to plant growth promotion (PGP).']\n",
      "[StringElement('e01237-22', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1128/mra.01237-22', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Genome Assembly of a Putative Plant Growth-Stimulating Bacterial Sweet Pepper Fruit Isolate, Enterobacter hormaechei SRU4.4.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Genome', attributes={'MajorTopicYN': 'N'}), StringElement('Mulberry bacterial wilt', attributes={'MajorTopicYN': 'N'}), StringElement('Pantoea ananatis', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36691281', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '01', 'Day': '24'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '01', 'Day': '23'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1094/PDIS-10-22-2473-A', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('0191-2917', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'PubDate': {'Year': '2023', 'Month': 'Jan', 'Day': '23'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Plant disease', 'ISOAbbreviation': 'Plant Dis'}, 'ArticleTitle': 'Complete Genome Sequence of Pantoea ananatis strain LCFJ-001, Isolated from Bacterial Wilt Mulberry.', 'Abstract': {'AbstractText': [\"A Pantoea ananatis strain, named LCFJ-001 (GDMCC: 1.6101), was isolated for the first time from bacterial wilt-diseased roots of mulberry (Morus atropurpurea) in the western part of the Guangxi Zhuang Autonomous Region, China. Moreover, through Koch's postulates, it was proven that LCFJ-001 can cause mulberry wilt, which is one of the pathogens of mulberry bacterial wilt. Here, we report a complete, annotated genome sequence of P. ananatis LCFJ-001. The entire genome sequence of P. ananatis strain LCFJ-001 was a 4,499,350 bp circular chromosome with 53.50% GC content. In total, 3521 genes were annotated, of which 3418 were assigned protein-coding genes. In addition, 22 ribosomal RNAs and 81 transfer RNAs were identified. The presented resource will help explore the pathogenetic mechanisms of mulberry wilt disease caused by the genus Pantoea.\"]}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agricultural University, School of Animal Sciences, School of Animal Science, South China Agricultural University, Wushan Street, Tianhe District, Guangzhou City, Guangdong Province, China, guangzhou, China, 510640; yuanting361@163.com.'}], 'Identifier': [], 'LastName': 'Yuan', 'ForeName': 'Ting', 'Initials': 'T'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agricultural University College of Animal Science, 527517, 483 Wushan Road, Tianhe District, Guangzhou, China, 510642; 1297981638@qq.com.'}], 'Identifier': [], 'LastName': 'Huang', 'ForeName': 'Yuxing', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agriculture University, College of Animal Science, South China Agriculture University, Wushan road, Guangzhou, Guangdong, 510642, China, Guangzhou, GUANGDONG, China, 510642; 1594500552@QQ.COM.'}], 'Identifier': [], 'LastName': 'Luo', 'ForeName': 'Longhui', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agricultural University College of Animal Science, 527517, Guangzhou, China; 853240266@qq.com.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Jicheng', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agricultural University College of Animal Science, 527517, Guangzhou, China; 10639934@qq.com.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Jinhao', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Guangzhou, China; chenjiehu@scgene.com.'}], 'Identifier': [], 'LastName': 'Chen', 'ForeName': 'Jiehu', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'He Chi Universiry, Hechi, China; 347220509@qq.com.'}], 'Identifier': [], 'LastName': 'Qin', 'ForeName': 'Yue', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'South China Agricultural University, 12526, College of Animal Science, Guangzhou, Guangdong, China; liujiping@scau.edu.cn.'}], 'Identifier': [], 'LastName': 'Liu', 'ForeName': 'Jiping', 'Initials': 'J'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'United States', 'MedlineTA': 'Plant Dis', 'NlmUniqueID': '9882809', 'ISSNLinking': '0191-2917'}}, attributes={'Status': 'Publisher', 'Owner': 'NLM'})\n",
      "[\"A Pantoea ananatis strain, named LCFJ-001 (GDMCC: 1.6101), was isolated for the first time from bacterial wilt-diseased roots of mulberry (Morus atropurpurea) in the western part of the Guangxi Zhuang Autonomous Region, China. Moreover, through Koch's postulates, it was proven that LCFJ-001 can cause mulberry wilt, which is one of the pathogens of mulberry bacterial wilt. Here, we report a complete, annotated genome sequence of P. ananatis LCFJ-001. The entire genome sequence of P. ananatis strain LCFJ-001 was a 4,499,350 bp circular chromosome with 53.50% GC content. In total, 3521 genes were annotated, of which 3418 were assigned protein-coding genes. In addition, 22 ribosomal RNAs and 81 transfer RNAs were identified. The presented resource will help explore the pathogenetic mechanisms of mulberry wilt disease caused by the genus Pantoea.\"]\n",
      "[StringElement('10.1094/PDIS-10-22-2473-A', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Complete Genome Sequence of Pantoea ananatis strain LCFJ-001, Isolated from Bacterial Wilt Mulberry.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Branchinella thailandensis', attributes={'MajorTopicYN': 'N'}), StringElement('Streptocephalus sirindhornae', attributes={'MajorTopicYN': 'N'}), StringElement('bacteria', attributes={'MajorTopicYN': 'N'}), StringElement('fairy shrimp', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36649095', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '01', 'Day': '19'}, 'DateRevised': {'Year': '2023', 'Month': '01', 'Day': '19'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [], 'ELocationID': [StringElement('10.24425/pjvs.2022.140857', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('2300-2557', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '25', 'Issue': '4', 'PubDate': {'Year': '2022', 'Month': 'Dec'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Polish journal of veterinary sciences', 'ISOAbbreviation': 'Pol J Vet Sci'}, 'ArticleTitle': 'Bacterial communities in Branchinella thailandensis and Streptocephalus sirindhornae.', 'Pagination': {'StartPage': '493', 'EndPage': '500', 'MedlinePgn': '493-500'}, 'Abstract': {'AbstractText': ['The fairy shrimp is a freshwater crustacean found in both temporary and permanent freshwaters. In Thailand, fairy shrimp are farmed as live food for ornamental fish. This study aimed to investigate the bacterial compositions in two fairy shrimp species, Branchinella thailandensis and Streptocephalus sirindhornae. Both species were cultured, and total DNA was extracted. The V3-V4 region of the 16S rRNA gene was amplified and sequenced using Illumina Miseq. All data were analyzed by Illumina 16S Metagenomics (version 1.0.1) workflow in Base Space- Illumina. Each read was blasted against the Illumina-curated version of the Greengenes database to determine the operational taxonomic units (OTUs) corresponding to the 16S rRNA gene sequence. The results showed that the Shanon-Weiner diversity index of bacterial compositions in B. thailandensis and S. sirindhornae were 2.135 and 3.122, respectively. The evenness and genus-level richness of the bacterial composition in B. thailandensis were 0.364 and 354 genera, respectively. The dominant bacterium found in B. thailandensis was Nevskia. In S. sirindhornae, the evenness and genus-level richness of the bacterial composition were 0.521 and 400 genera, respectively. Azohydromonas was the dominant bacterium. Our results showed that the compositions and proportions of bacterial communities were specific to each species of fairy shrimp. This study will be useful for further experiments in aquaculture and ecological studies related to symbiotic interaction.'], 'CopyrightInformation': 'Copyright© by the Polish Academy of Sciences.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Division of Biological Science, Faculty of Science, Prince of Songkla University, Hat Yai, Songkhla 90110, Thailand.'}, {'Identifier': [], 'Affiliation': 'Gut Biology and Microbiota Research Unit, Faculty of Science, Prince of Songkla University, Hat Yai, Songkhla 90110, Thailand.'}], 'Identifier': [], 'LastName': 'Peerakietkhajorn', 'ForeName': 'S', 'Initials': 'S'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Division of Biological Science, Faculty of Science, Prince of Songkla University, Hat Yai, Songkhla 90110, Thailand.'}], 'Identifier': [], 'LastName': 'Sinso', 'ForeName': 'T', 'Initials': 'T'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print'}), 'MedlineJournalInfo': {'Country': 'Germany', 'MedlineTA': 'Pol J Vet Sci', 'NlmUniqueID': '101125473', 'ISSNLinking': '1505-1773'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('DNA, Bacterial', attributes={'UI': 'D004269'})}], 'MeshHeadingList': [{'QualifierName': [], 'DescriptorName': StringElement('Animals', attributes={'UI': 'D000818', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'}), StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Anostraca', attributes={'UI': 'D033202', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Thailand', attributes={'UI': 'D013785', 'MajorTopicYN': 'N', 'Type': 'Geographic'})}, {'QualifierName': [], 'DescriptorName': StringElement('Phylogeny', attributes={'UI': 'D010802', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('DNA, Bacterial', attributes={'UI': 'D004269', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['The fairy shrimp is a freshwater crustacean found in both temporary and permanent freshwaters. In Thailand, fairy shrimp are farmed as live food for ornamental fish. This study aimed to investigate the bacterial compositions in two fairy shrimp species, Branchinella thailandensis and Streptocephalus sirindhornae. Both species were cultured, and total DNA was extracted. The V3-V4 region of the 16S rRNA gene was amplified and sequenced using Illumina Miseq. All data were analyzed by Illumina 16S Metagenomics (version 1.0.1) workflow in Base Space- Illumina. Each read was blasted against the Illumina-curated version of the Greengenes database to determine the operational taxonomic units (OTUs) corresponding to the 16S rRNA gene sequence. The results showed that the Shanon-Weiner diversity index of bacterial compositions in B. thailandensis and S. sirindhornae were 2.135 and 3.122, respectively. The evenness and genus-level richness of the bacterial composition in B. thailandensis were 0.364 and 354 genera, respectively. The dominant bacterium found in B. thailandensis was Nevskia. In S. sirindhornae, the evenness and genus-level richness of the bacterial composition were 0.521 and 400 genera, respectively. Azohydromonas was the dominant bacterium. Our results showed that the compositions and proportions of bacterial communities were specific to each species of fairy shrimp. This study will be useful for further experiments in aquaculture and ecological studies related to symbiotic interaction.']\n",
      "[StringElement('10.24425/pjvs.2022.140857', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Bacterial communities in Branchinella thailandensis and Streptocephalus sirindhornae.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Bacillus cereus, hydrocarbon contamination', attributes={'MajorTopicYN': 'N'}), StringElement('Bacillus pumilus', attributes={'MajorTopicYN': 'N'}), StringElement('Biosurfactant', attributes={'MajorTopicYN': 'N'}), StringElement('PAHs', attributes={'MajorTopicYN': 'N'}), StringElement('TPH', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36587720', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '02', 'Day': '07'}, 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '27'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '29'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1016/j.envres.2022.115092', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S0013-9351(22)02419-7', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1096-0953', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '220', 'PubDate': {'Year': '2023', 'Month': 'Mar', 'Day': '01'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Environmental research', 'ISOAbbreviation': 'Environ Res'}, 'ArticleTitle': 'Soil treatment using a biosurfactant producing bacterial consortium in rice fields contaminated with oily sludge- a sustainable approach.', 'Pagination': {'StartPage': '115092', 'MedlinePgn': '115092'}, 'Abstract': {'AbstractText': [\"A consortium of two biosurfactant-producing bacteria (Bacillus pumilus KS2 and Bacillus cereus R2) was developed to remediate petroleum hydrocarbon-contaminated paddy soil. Soil samples from a heavily contaminated rice field near Assam's Lakwa oilfield were collected and placed in earthen pots for treatment. After each month of incubation, 50\\xa0g of soil from each earthen pot was collected, and the soil TPH (ppm) in each sample was determined. The extracted TPH samples were analysed by Gas chromatography-mass spectrometry (GC-MS) to confirm microbial degradation. The soil samples were examined for changes in pH, conductivity, total organic content (TOC), water holding capacity, and total nitrogen content in addition to TPH degradation. An increasing trend in TPH degradation was observed with each passing month. After six months of treatment, the sample with the lowest initial TPH concentration (1735\\xa0ppm) had the highest degradation (91.24%), while the soil with the highest amount of TPH (5780\\xa0ppm) had the lowest degradation (74.35%). A wide range of aliphatic hydrocarbons found in soil samples was degraded by the bacterial consortium. The soil samples contained eight different low- and high-molecular-weight PAHs. Some were fully mineralized, while others were significantly reduced. With the decrease in the TPH level in the polluted soil, a significant improvement in the soil's physicochemical qualities (such as pH, electrical conductivity, total organic content, and water-holding capacity) was observed.\"], 'CopyrightInformation': 'Copyright © 2022 Elsevier Inc. All rights reserved.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Applied Biology, School of Biological Sciences, University of Science and Technology Meghalaya (USTM), 9th Mile, Techno City, Baridua, Ri-Bhoi, 793101, Meghalaya, India; Environmental Biotechnology Laboratory, Life Sciences Division, Institute of Advanced Study in Science & Technology (IASST), Paschim Boragaon, Guwahati, 781 035, Assam, India.'}], 'Identifier': [], 'LastName': 'Patowary', 'ForeName': 'Kaustuvmani', 'Initials': 'K'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Applied Biology, School of Biological Sciences, University of Science and Technology Meghalaya (USTM), 9th Mile, Techno City, Baridua, Ri-Bhoi, 793101, Meghalaya, India.'}], 'Identifier': [], 'LastName': 'Bhuyan', 'ForeName': 'Tamanna', 'Initials': 'T'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Environmental Biotechnology Laboratory, Life Sciences Division, Institute of Advanced Study in Science & Technology (IASST), Paschim Boragaon, Guwahati, 781 035, Assam, India.'}], 'Identifier': [], 'LastName': 'Patowary', 'ForeName': 'Rupshikha', 'Initials': 'R'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Applied Biology, School of Biological Sciences, University of Science and Technology Meghalaya (USTM), 9th Mile, Techno City, Baridua, Ri-Bhoi, 793101, Meghalaya, India.'}], 'Identifier': [], 'LastName': 'Mohanta', 'ForeName': 'Yugal Kishore', 'Initials': 'YK'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Salim Ali Center for Ornithology and Natural History, Coimbatore, 641108, India.'}], 'Identifier': [], 'LastName': 'Panda', 'ForeName': 'Bibhu Prasad', 'Initials': 'BP'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Environmental Biotechnology Laboratory, Life Sciences Division, Institute of Advanced Study in Science & Technology (IASST), Paschim Boragaon, Guwahati, 781 035, Assam, India.'}], 'Identifier': [], 'LastName': 'Deka', 'ForeName': 'Suresh', 'Initials': 'S'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institutional Biotech Hub (IBT Hub), Department of Botany, NN Saikia College, Titabar, 785630, Assam, India.'}], 'Identifier': [], 'LastName': 'Islam', 'ForeName': 'Nazim Forid', 'Initials': 'NF'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Oil & Gas Research Center, Central Analytical and Applied Research Unit, Sultan Qaboos University, Oman.'}], 'Identifier': [], 'LastName': 'Joshi', 'ForeName': 'Sanket J', 'Initials': 'SJ'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Bioremediation Technology Research Group, Department of Botany, Bodoland University, Rangalikhata, Deborgaon, 783370, Kokrajhar (BTR), Assam, India. Electronic address: hemen@buniv.edu.in.'}], 'Identifier': [], 'LastName': 'Sarma', 'ForeName': 'Hemen', 'Initials': 'H'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'}), StringElement(\"Research Support, Non-U.S. Gov't\", attributes={'UI': 'D013485'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'Netherlands', 'MedlineTA': 'Environ Res', 'NlmUniqueID': '0147621', 'ISSNLinking': '0013-9351'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('Sewage', attributes={'UI': 'D012722'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Soil', attributes={'UI': 'D012987'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Soil Pollutants', attributes={'UI': 'D012989'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Hydrocarbons', attributes={'UI': 'D006838'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Petroleum', attributes={'UI': 'D010578'})}], 'MeshHeadingList': [{'QualifierName': [StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Sewage', attributes={'UI': 'D012722', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Oryza', attributes={'UI': 'D012275', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('chemistry', attributes={'UI': 'Q000737', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Soil', attributes={'UI': 'D012987', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Biodegradation, Environmental', attributes={'UI': 'D001673', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('analysis', attributes={'UI': 'Q000032', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Soil Pollutants', attributes={'UI': 'D012989', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Soil Microbiology', attributes={'UI': 'D012988', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Hydrocarbons', attributes={'UI': 'D006838', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('analysis', attributes={'UI': 'Q000032', 'MajorTopicYN': 'N'}), StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Petroleum', attributes={'UI': 'D010578', 'MajorTopicYN': 'Y'})}], 'CoiStatement': 'Declaration of competing interest The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.'}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "[\"A consortium of two biosurfactant-producing bacteria (Bacillus pumilus KS2 and Bacillus cereus R2) was developed to remediate petroleum hydrocarbon-contaminated paddy soil. Soil samples from a heavily contaminated rice field near Assam's Lakwa oilfield were collected and placed in earthen pots for treatment. After each month of incubation, 50\\xa0g of soil from each earthen pot was collected, and the soil TPH (ppm) in each sample was determined. The extracted TPH samples were analysed by Gas chromatography-mass spectrometry (GC-MS) to confirm microbial degradation. The soil samples were examined for changes in pH, conductivity, total organic content (TOC), water holding capacity, and total nitrogen content in addition to TPH degradation. An increasing trend in TPH degradation was observed with each passing month. After six months of treatment, the sample with the lowest initial TPH concentration (1735\\xa0ppm) had the highest degradation (91.24%), while the soil with the highest amount of TPH (5780\\xa0ppm) had the lowest degradation (74.35%). A wide range of aliphatic hydrocarbons found in soil samples was degraded by the bacterial consortium. The soil samples contained eight different low- and high-molecular-weight PAHs. Some were fully mineralized, while others were significantly reduced. With the decrease in the TPH level in the polluted soil, a significant improvement in the soil's physicochemical qualities (such as pH, electrical conductivity, total organic content, and water-holding capacity) was observed.\"]\n",
      "[StringElement('10.1016/j.envres.2022.115092', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S0013-9351(22)02419-7', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})]\n",
      "Soil treatment using a biosurfactant producing bacterial consortium in rice fields contaminated with oily sludge- a sustainable approach.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Bacillus pseudomycoides', attributes={'MajorTopicYN': 'N'}), StringElement('Polyhydroxybutyrate', attributes={'MajorTopicYN': 'N'}), StringElement('Saline conditions', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36563813', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '02', 'Day': '03'}, 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '03'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '21'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1016/j.ijbiomac.2022.12.190', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S0141-8130(22)03101-4', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1879-0003', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '229', 'PubDate': {'Year': '2023', 'Month': 'Feb', 'Day': '28'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'International journal of biological macromolecules', 'ISOAbbreviation': 'Int J Biol Macromol'}, 'ArticleTitle': 'In-silico analysis of a halophilic bacterial isolate-Bacillus pseudomycoides SAS-B1 and its polyhydroxybutyrate production through fed-batch approach under differential salt conditions.', 'Pagination': {'StartPage': '372', 'EndPage': '387', 'MedlinePgn': '372-387'}, 'Abstract': {'AbstractText': ['Polyhydroxybutyrate (PHB) is a natural biopolymer and a viable substitute for petroleum-derived polymers that possess immense potential for diverse applications. In the present study, PHB was produced by a halophilic bacteria identified as Bacillus pseudomycoides SAS-B1 by 16S rRNA gene sequencing. The bacterial genome was evaluated through complete genome sequencing, which elucidated a 5,338,308\\xa0bp genome with 34.88\\xa0% of G\\xa0+\\xa0C content and 5660 genes. Other genome attributes were analyzed such as functional profiling, gene ontology, and metabolic pathways. Genes involved in PHB biochemical pathway were identified such as phaA, phaB, and phaC. Furthermore, sodium-dependent transporters and other ATP-binding genes were identified in the genome that may be involved in sodium uptake during saline conditions. The PHB production by B. pseudomycoides SAS-B1 was examined under differential salt conditions. The PHB yield was increased from 3.14\\xa0±\\xa00.02\\xa0g/L to 6.12\\xa0±\\xa00.04\\xa0g/L when salinity was increased upto 20\\xa0g/L with intermittent feeding of glucose and corn steep liquor. FTIR, NMR, and GC-MS studies elucidated the presence of desired functional groups, molecular structure, and monomeric compositions of PHB respectively. Further, TGA revealed the thermal stability of the recovered PHB upto (220-230)\\xa0°C and has a crystallinity index of upto 33\\xa0±\\xa00.5\\xa0% as confirmed by XRD analysis.'], 'CopyrightInformation': 'Copyright © 2022 Elsevier B.V. All rights reserved.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Division of Chemical Engineering, Centre of Innovative and Applied Bioprocessing, Knowledge City, Sector-81, Mohali, Punjab 140306, India.'}], 'Identifier': [], 'LastName': 'Anjana'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Division of Chemical Engineering, Centre of Innovative and Applied Bioprocessing, Knowledge City, Sector-81, Mohali, Punjab 140306, India.'}], 'Identifier': [], 'LastName': 'Rawat', 'ForeName': 'Shristhi', 'Initials': 'S'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Division of Chemical Engineering, Centre of Innovative and Applied Bioprocessing, Knowledge City, Sector-81, Mohali, Punjab 140306, India. Electronic address: saswatagoswami2015@gmail.com.'}], 'Identifier': [], 'LastName': 'Goswami', 'ForeName': 'Saswata', 'Initials': 'S'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'Netherlands', 'MedlineTA': 'Int J Biol Macromol', 'NlmUniqueID': '7909578', 'ISSNLinking': '0141-8130'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('Polyesters', attributes={'UI': 'D011091'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('polyhydroxybutyrate', attributes={'UI': 'C000720856'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': '451W47IQ8X', 'NameOfSubstance': StringElement('Sodium Chloride', attributes={'UI': 'D012965'})}, {'RegistryNumber': '9NEZ333N27', 'NameOfSubstance': StringElement('Sodium', attributes={'UI': 'D012964'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Hydroxybutyrates', attributes={'UI': 'D006885'})}], 'SupplMeshList': [StringElement('Bacillus pseudomycoides', attributes={'Type': 'Organism', 'UI': 'C000650143'})], 'MeshHeadingList': [{'QualifierName': [StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Polyesters', attributes={'UI': 'D011091', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('pharmacology', attributes={'UI': 'Q000494', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Sodium Chloride', attributes={'UI': 'D012965', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Sodium', attributes={'UI': 'D012964', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('chemistry', attributes={'UI': 'Q000737', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Hydroxybutyrates', attributes={'UI': 'D006885', 'MajorTopicYN': 'N'})}], 'CoiStatement': 'Declaration of competing interest The authors declare that they have no known competing financial or personal interests.'}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['Polyhydroxybutyrate (PHB) is a natural biopolymer and a viable substitute for petroleum-derived polymers that possess immense potential for diverse applications. In the present study, PHB was produced by a halophilic bacteria identified as Bacillus pseudomycoides SAS-B1 by 16S rRNA gene sequencing. The bacterial genome was evaluated through complete genome sequencing, which elucidated a 5,338,308\\xa0bp genome with 34.88\\xa0% of G\\xa0+\\xa0C content and 5660 genes. Other genome attributes were analyzed such as functional profiling, gene ontology, and metabolic pathways. Genes involved in PHB biochemical pathway were identified such as phaA, phaB, and phaC. Furthermore, sodium-dependent transporters and other ATP-binding genes were identified in the genome that may be involved in sodium uptake during saline conditions. The PHB production by B. pseudomycoides SAS-B1 was examined under differential salt conditions. The PHB yield was increased from 3.14\\xa0±\\xa00.02\\xa0g/L to 6.12\\xa0±\\xa00.04\\xa0g/L when salinity was increased upto 20\\xa0g/L with intermittent feeding of glucose and corn steep liquor. FTIR, NMR, and GC-MS studies elucidated the presence of desired functional groups, molecular structure, and monomeric compositions of PHB respectively. Further, TGA revealed the thermal stability of the recovered PHB upto (220-230)\\xa0°C and has a crystallinity index of upto 33\\xa0±\\xa00.5\\xa0% as confirmed by XRD analysis.']\n",
      "[StringElement('10.1016/j.ijbiomac.2022.12.190', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}), StringElement('S0141-8130(22)03101-4', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})]\n",
      "In-silico analysis of a halophilic bacterial isolate-Bacillus pseudomycoides SAS-B1 and its polyhydroxybutyrate production through fed-batch approach under differential salt conditions.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36536230', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2022', 'Month': '12', 'Day': '21'}, 'DateRevised': {'Year': '2022', 'Month': '12', 'Day': '22'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '19'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1007/s00284-022-03155-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1432-0991', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '80', 'Issue': '1', 'PubDate': {'Year': '2022', 'Month': 'Dec', 'Day': '19'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Current microbiology', 'ISOAbbreviation': 'Curr Microbiol'}, 'ArticleTitle': 'Lysobacter lactosilyticus sp. nov., a Novel β-Galactosidase Producing Bacterial Strain Isolated from Farmland Soil Applied with Amino Acid Fertilizer.', 'Pagination': {'StartPage': '43', 'MedlinePgn': '43'}, 'Abstract': {'AbstractText': ['To isolate β-galactosidase producing bacterial resources, a novel Gram-stain-negative, strictly aerobic bacterial strain designated as A6<sup>T</sup> was obtained from a farmland soil sample. Cells of the strain were rod-shaped (0.4-0.7\\xa0μm\\u2009×\\u20091.8-2.2\\xa0μm) without flagella and motility. Strain A6<sup>T</sup> grew optimally at 30 °C, pH 7.0 with 0% (w/v) NaCl. Based on phylogenetic analysis, strain A6<sup>T</sup> clustered within the genus Lysobacter clade and branched with Lysobacter dokdonensis KCTC 12822<sup>T</sup> (99.5%, 16S rRNA gene sequence similarity) and Lysobacter caseinilyticus KACC 19816<sup>T</sup> (98.5%). The average nucleotide identity (ANI) and digital DNA-DNA hybridization (dDDH) values between strain A6<sup>T</sup> and Lysobacter dokdonensis KCTC 12822<sup>T</sup> were 82.7% and 26.2%, and the values for strain A6<sup>T</sup> and KACC 19816<sup>T</sup> were 81.4% and 23.8%, respectively. Iso-C<sub>16:0</sub>, iso-C<sub>15:0</sub>, summed feature 9 (C<sub>17:1</sub> iso ω9c and/or C<sub>16:0</sub> 10-methyl) and summed feature 3 (C<sub>16:1</sub>ω7c and/or C<sub>16:1</sub> ω6c) were the major fatty acids, diphosphatidylglycerol, phosphatidylglycerol, and phosphatidylethanolamine were the major polar lipids, and ubiquinone 8 (Q-8) was the major ubiquinone. The genomic DNA G+C content was 67.2\\xa0mol%. Furthermore, under the condition of 30 °C, pH 7.0, 4% inoculation with 10.0\\xa0g\\xa0L<sup>-1</sup> lactose, the β-galactosidase activity produced by strain A6<sup>T</sup> was highest, reaching 95.3 U\\xa0mL<sup>-1</sup>, indicating that this strain could be applied as a potential strain for β-galactosidase production. Strain A6<sup>T</sup> represents a novel species of the genus Lysobacter, and Lysobacter lactosilyticus sp. nov. is proposed on the basis of phenotypic, genotypic, and chemotaxonomic analysis. The type strain is A6<sup>T</sup> (=KCTC 82184<sup>T</sup>=CGMCC 1.18582<sup>T</sup>).'], 'CopyrightInformation': '© 2022. The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource and Environment, Anhui Science and Technology University, Fengyang, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Xiang', 'Initials': 'X'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource and Environment, Anhui Science and Technology University, Fengyang, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Yu-Huan', 'Initials': 'YH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource and Environment, Anhui Science and Technology University, Fengyang, China.'}], 'Identifier': [], 'LastName': 'Yang', 'ForeName': 'Hong-Xing', 'Initials': 'HX'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource and Environment, Anhui Science and Technology University, Fengyang, China.'}], 'Identifier': [], 'LastName': 'Chuang', 'ForeName': 'Shao-Chuang', 'Initials': 'SC'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource and Environment, Anhui Science and Technology University, Fengyang, China.'}], 'Identifier': [], 'LastName': 'Zhou', 'ForeName': 'Chun-Fu', 'Initials': 'CF'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Centre for Carbon, Water and Food, School of Life and Environmental Sciences, The University of Sydney, Camden, NSW, 2570, Australia.'}], 'Identifier': [], 'LastName': 'Yu', 'ForeName': 'Xing', 'Initials': 'X'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': \"School of Life Science and Agricultural Engineering, Research Center of Henan Provincial Agricultural Biomass Resource Engineering and Technology, Henan Province Artemisiae Argyi Development and Utilization Engineering Technology Research Center, Innovation Center of Water Security for Water Source Region of Mid-Route Project of South-North Water Diversion of Henan Province, Nanyang Normal University, Nanyang, 473061, People's Republic of China. zhanghao660@nynu.edu.cn.\"}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Hao', 'Initials': 'H'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic'}), 'MedlineJournalInfo': {'Country': 'United States', 'MedlineTA': 'Curr Microbiol', 'NlmUniqueID': '7808448', 'ISSNLinking': '0343-8651'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('Phospholipids', attributes={'UI': 'D010743'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Fertilizers', attributes={'UI': 'D005308'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Soil', attributes={'UI': 'D012987'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Amino Acids', attributes={'UI': 'D000596'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('DNA, Bacterial', attributes={'UI': 'D004269'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Fatty Acids', attributes={'UI': 'D005227'})}, {'RegistryNumber': 'EC 3.2.1.23', 'NameOfSubstance': StringElement('beta-Galactosidase', attributes={'UI': 'D001616'})}], 'SupplMeshList': [StringElement('Lysobacter dokdonensis', attributes={'Type': 'Organism', 'UI': 'C000641728'})], 'MeshHeadingList': [{'QualifierName': [StringElement('chemistry', attributes={'UI': 'Q000737', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Phospholipids', attributes={'UI': 'D010743', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Lysobacter', attributes={'UI': 'D053528', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('analysis', attributes={'UI': 'Q000032', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Fertilizers', attributes={'UI': 'D005308', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Phylogeny', attributes={'UI': 'D010802', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Soil', attributes={'UI': 'D012987', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('metabolism', attributes={'UI': 'Q000378', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Amino Acids', attributes={'UI': 'D000596', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Farms', attributes={'UI': 'D000072480', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('DNA, Bacterial', attributes={'UI': 'D004269', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Soil Microbiology', attributes={'UI': 'D012988', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('chemistry', attributes={'UI': 'Q000737', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Fatty Acids', attributes={'UI': 'D005227', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('beta-Galactosidase', attributes={'UI': 'D001616', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Sequence Analysis, DNA', attributes={'UI': 'D017422', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacterial Typing Techniques', attributes={'UI': 'D015373', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['To isolate β-galactosidase producing bacterial resources, a novel Gram-stain-negative, strictly aerobic bacterial strain designated as A6<sup>T</sup> was obtained from a farmland soil sample. Cells of the strain were rod-shaped (0.4-0.7\\xa0μm\\u2009×\\u20091.8-2.2\\xa0μm) without flagella and motility. Strain A6<sup>T</sup> grew optimally at 30 °C, pH 7.0 with 0% (w/v) NaCl. Based on phylogenetic analysis, strain A6<sup>T</sup> clustered within the genus Lysobacter clade and branched with Lysobacter dokdonensis KCTC 12822<sup>T</sup> (99.5%, 16S rRNA gene sequence similarity) and Lysobacter caseinilyticus KACC 19816<sup>T</sup> (98.5%). The average nucleotide identity (ANI) and digital DNA-DNA hybridization (dDDH) values between strain A6<sup>T</sup> and Lysobacter dokdonensis KCTC 12822<sup>T</sup> were 82.7% and 26.2%, and the values for strain A6<sup>T</sup> and KACC 19816<sup>T</sup> were 81.4% and 23.8%, respectively. Iso-C<sub>16:0</sub>, iso-C<sub>15:0</sub>, summed feature 9 (C<sub>17:1</sub> iso ω9c and/or C<sub>16:0</sub> 10-methyl) and summed feature 3 (C<sub>16:1</sub>ω7c and/or C<sub>16:1</sub> ω6c) were the major fatty acids, diphosphatidylglycerol, phosphatidylglycerol, and phosphatidylethanolamine were the major polar lipids, and ubiquinone 8 (Q-8) was the major ubiquinone. The genomic DNA G+C content was 67.2\\xa0mol%. Furthermore, under the condition of 30 °C, pH 7.0, 4% inoculation with 10.0\\xa0g\\xa0L<sup>-1</sup> lactose, the β-galactosidase activity produced by strain A6<sup>T</sup> was highest, reaching 95.3 U\\xa0mL<sup>-1</sup>, indicating that this strain could be applied as a potential strain for β-galactosidase production. Strain A6<sup>T</sup> represents a novel species of the genus Lysobacter, and Lysobacter lactosilyticus sp. nov. is proposed on the basis of phenotypic, genotypic, and chemotaxonomic analysis. The type strain is A6<sup>T</sup> (=KCTC 82184<sup>T</sup>=CGMCC 1.18582<sup>T</sup>).']\n",
      "[StringElement('10.1007/s00284-022-03155-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Lysobacter lactosilyticus sp. nov., a Novel β-Galactosidase Producing Bacterial Strain Isolated from Farmland Soil Applied with Amino Acid Fertilizer.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Harmonia axyridis', attributes={'MajorTopicYN': 'N'}), StringElement('high-throughput sequencing', attributes={'MajorTopicYN': 'N'}), StringElement('life stages', attributes={'MajorTopicYN': 'N'}), StringElement('quantitative PCR', attributes={'MajorTopicYN': 'N'}), StringElement('symbiotic bacteria', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36532478', attributes={'Version': '1'}), 'DateRevised': {'Year': '2022', 'Month': '12', 'Day': '22'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '01'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('1050329', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1050329', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1664-302X', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '13', 'PubDate': {'Year': '2022'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Frontiers in microbiology', 'ISOAbbreviation': 'Front Microbiol'}, 'ArticleTitle': 'Dynamics of symbiotic bacterial community in whole life stage of <i>Harmonia axyridis</i> (Coleoptera: Coccinellidae).', 'Pagination': {'StartPage': '1050329', 'MedlinePgn': '1050329'}, 'Abstract': {'AbstractText': [StringElement('Bacteria play critical roles in the reproduction, metabolism, physiology, and detoxification of their insect hosts. The ladybird beetle (<i>Harmonia axyridis</i>) harbors a myriad of endosymbiotic microbes. However, to date, little is known about how the microbial composition of <i>H. axyridis</i> varies throughout its life cycle.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('In this study, 16S rRNA amplicon sequencing and quantitative PCR were employed to investigate the diversity and dynamics of bacterial symbionts across the egg, larval, pupae, and adults stages of <i>H. axyridis</i>.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('Higher bacterial community richness and diversity were observed in eggs, followed by those in adults and pupae. The community richness index differed significantly between second-instar larvae and other developmental stages. Proteobacteria, Firmicutes, and Actinobacteria were the dominant phyla. <i>Staphylococcus, Enterobacter, Glutamicibacter</i>, and <i>Acinetobacter</i> were the dominant bacteria genera; however, their relative abundances fluctuated across host developmental stages. Interestingly, the larval stage harbored high proportions of Firmicutes, whereas the adult microbial community largely consisted of Proteobacteria.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('This study is the first to determine the symbiotic bacterial composition across key life stages of <i>H. axyridis</i>. These outcomes can foster the development of environmental risk assessments and novel biological control strategies.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})], 'CopyrightInformation': 'Copyright © 2022 Du, Xue, Hu, Zhu, Wang, Zhang, Li, Ji, Niu, Luo, Cui and Gao.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}, {'Identifier': [], 'Affiliation': 'Hubei Insect Resources Utilization and Sustainable Pest Management Key Laboratory, Huazhong Agricultural University, Wuhan, China.'}], 'Identifier': [], 'LastName': 'Du', 'ForeName': 'Lingen', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Hubei Insect Resources Utilization and Sustainable Pest Management Key Laboratory, Huazhong Agricultural University, Wuhan, China.'}], 'Identifier': [], 'LastName': 'Xue', 'ForeName': 'Hui', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Hubei Insect Resources Utilization and Sustainable Pest Management Key Laboratory, Huazhong Agricultural University, Wuhan, China.'}], 'Identifier': [], 'LastName': 'Hu', 'ForeName': 'Fangmei', 'Initials': 'F'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Zhu', 'ForeName': 'Xiangzhen', 'Initials': 'X'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Li', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Kaixin', 'Initials': 'K'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Dongyang', 'Initials': 'D'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Ji', 'ForeName': 'Jichao', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Niu', 'ForeName': 'Lin', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Luo', 'ForeName': 'Junyu', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Cui', 'ForeName': 'Jinjie', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Cotton Biology, Institute of Cotton Research, Chinese Academy of Agricultural Sciences, Anyang, China.'}, {'Identifier': [], 'Affiliation': 'Zhengzhou Research Base, State Key Laboratory of Cotton Biology, School of Agricultural Sciences, Zhengzhou University, Zhengzhou, China.'}], 'Identifier': [], 'LastName': 'Gao', 'ForeName': 'Xueke', 'Initials': 'X'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Switzerland', 'MedlineTA': 'Front Microbiol', 'NlmUniqueID': '101548977', 'ISSNLinking': '1664-302X'}, 'CoiStatement': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "[StringElement('Bacteria play critical roles in the reproduction, metabolism, physiology, and detoxification of their insect hosts. The ladybird beetle (<i>Harmonia axyridis</i>) harbors a myriad of endosymbiotic microbes. However, to date, little is known about how the microbial composition of <i>H. axyridis</i> varies throughout its life cycle.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('In this study, 16S rRNA amplicon sequencing and quantitative PCR were employed to investigate the diversity and dynamics of bacterial symbionts across the egg, larval, pupae, and adults stages of <i>H. axyridis</i>.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('Higher bacterial community richness and diversity were observed in eggs, followed by those in adults and pupae. The community richness index differed significantly between second-instar larvae and other developmental stages. Proteobacteria, Firmicutes, and Actinobacteria were the dominant phyla. <i>Staphylococcus, Enterobacter, Glutamicibacter</i>, and <i>Acinetobacter</i> were the dominant bacteria genera; however, their relative abundances fluctuated across host developmental stages. Interestingly, the larval stage harbored high proportions of Firmicutes, whereas the adult microbial community largely consisted of Proteobacteria.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('This study is the first to determine the symbiotic bacterial composition across key life stages of <i>H. axyridis</i>. These outcomes can foster the development of environmental risk assessments and novel biological control strategies.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})]\n",
      "[StringElement('1050329', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1050329', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Dynamics of symbiotic bacterial community in whole life stage of <i>Harmonia axyridis</i> (Coleoptera: Coccinellidae).\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('alkaline phosphatase', attributes={'MajorTopicYN': 'N'}), StringElement('co-occurrence network', attributes={'MajorTopicYN': 'N'}), StringElement('keystone taxa', attributes={'MajorTopicYN': 'N'}), StringElement('long-term P fertilization', attributes={'MajorTopicYN': 'N'}), StringElement('organic P speciation', attributes={'MajorTopicYN': 'N'}), StringElement('phoD-harboring bacteria', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36504812', attributes={'Version': '1'}), 'DateRevised': {'Year': '2022', 'Month': '12', 'Day': '20'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '11', 'Day': '24'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('1045919', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1045919', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1664-302X', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '13', 'PubDate': {'Year': '2022'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Frontiers in microbiology', 'ISOAbbreviation': 'Front Microbiol'}, 'ArticleTitle': '<i>phoD</i>-harboring bacterial community composition dominates organic P mineralization under long-term P fertilization in acid purple soil.', 'Pagination': {'StartPage': '1045919', 'MedlinePgn': '1045919'}, 'Abstract': {'AbstractText': [StringElement('A better understanding of the regulatory role of microorganisms on soil phosphorous (P) mobilization is critical for developing sustainable fertilization practices and reducing P resource scarcity. The <i>phoD</i> genes regulate soil organic P (Po) mobilization.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('Based on the long-term P application experiments in acid purple soil of maize system in Southwest China (started in 2010), the experiment included five P levels: 0, 16, 33, 49, and 65.5 kg P hm<sup>-2</sup> (P0, P16, P33, P49, and P65.5, respectively). The molecular speciation of organic P in soil was determined by 31P-nuclear magnetic resonance (NMR), high-throughput sequencing technology, and real-time qPCR were used to analyze the bacterial community and abundance of <i>phoD</i>-harboring bacterial genes, exploring the bacterial community and abundance characteristics of <i>phoD</i> gene and its relationship with the forms of Po and alkaline phosphatase (ALP) activity in the soil.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The results showed that the orthophosphate monoesters (OM) were the main Po speciation and varied by P fertilization in acid purple soil. ALP activity decreased as P fertilization increased. Co-occurrence network analysis identified the overall network under five P fertilizations. The keystone taxon base on the network showed that <i>Collimonas</i>, <i>Roseateles</i>, <i>Mesorhizobium</i>, and <i>Cellulomonas</i> positively correlated with both OM and Po. The random forest showed that <i>Cellulomonas</i>, <i>Roseateles</i>, and <i>Rhodoplanes</i> were the key predictors for ALP activity. The keystone taxon was a more important predictor than the dominant taxon for ALP, OM, and Po. The structural equation model (SEM) showed that soil organic matter (SOM), available P (AP), and OM were the main factors influencing the ALP by reshaping <i>phoD</i>-harboring bacteria alpha diversity, community composition, and <i>phoD</i> abundance.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The <i>phoD</i>-harboring bacterial community composition especially the keystone taxon rather than alpha diversity and abundance dominated the ALP activity, which could promote P utilization over an intensive agroecosystem. These findings improve the understanding of how long-term gradient fertilization influences the community composition and function of P-solubilizing microorganisms in acid purple soil.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})], 'CopyrightInformation': 'Copyright © 2022 Lang, Li, Lakshmanan, Chen and Chen.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resources and Environment, Academy of Agricultural Sciences, Southwest University, Chongqing, China.'}, {'Identifier': [], 'Affiliation': 'Interdisciplinary Research Center for Agriculture Green Development in Yangtze River Basin, Southwest University, Chongqing, China.'}], 'Identifier': [], 'LastName': 'Lang', 'ForeName': 'Ming', 'Initials': 'M'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resources and Environment, Academy of Agricultural Sciences, Southwest University, Chongqing, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Haoming', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Interdisciplinary Research Center for Agriculture Green Development in Yangtze River Basin, Southwest University, Chongqing, China.'}, {'Identifier': [], 'Affiliation': 'Sugarcane Research Institute, Guangxi Academy of Agricultural Sciences, Nanning, China.'}, {'Identifier': [], 'Affiliation': 'Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, St Lucia, QLD, Australia.'}], 'Identifier': [], 'LastName': 'Lakshmanan', 'ForeName': 'Prakash', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resource Sciences, Sichuan Agricultural University, Chengdu, China.'}], 'Identifier': [], 'LastName': 'Chen', 'ForeName': 'Yuanxue', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'College of Resources and Environment, Academy of Agricultural Sciences, Southwest University, Chongqing, China.'}, {'Identifier': [], 'Affiliation': 'Interdisciplinary Research Center for Agriculture Green Development in Yangtze River Basin, Southwest University, Chongqing, China.'}], 'Identifier': [], 'LastName': 'Chen', 'ForeName': 'Xinping', 'Initials': 'X'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Switzerland', 'MedlineTA': 'Front Microbiol', 'NlmUniqueID': '101548977', 'ISSNLinking': '1664-302X'}, 'CoiStatement': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "[StringElement('A better understanding of the regulatory role of microorganisms on soil phosphorous (P) mobilization is critical for developing sustainable fertilization practices and reducing P resource scarcity. The <i>phoD</i> genes regulate soil organic P (Po) mobilization.', attributes={'Label': 'INTRODUCTION', 'NlmCategory': 'UNASSIGNED'}), StringElement('Based on the long-term P application experiments in acid purple soil of maize system in Southwest China (started in 2010), the experiment included five P levels: 0, 16, 33, 49, and 65.5 kg P hm<sup>-2</sup> (P0, P16, P33, P49, and P65.5, respectively). The molecular speciation of organic P in soil was determined by 31P-nuclear magnetic resonance (NMR), high-throughput sequencing technology, and real-time qPCR were used to analyze the bacterial community and abundance of <i>phoD</i>-harboring bacterial genes, exploring the bacterial community and abundance characteristics of <i>phoD</i> gene and its relationship with the forms of Po and alkaline phosphatase (ALP) activity in the soil.', attributes={'Label': 'METHODS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The results showed that the orthophosphate monoesters (OM) were the main Po speciation and varied by P fertilization in acid purple soil. ALP activity decreased as P fertilization increased. Co-occurrence network analysis identified the overall network under five P fertilizations. The keystone taxon base on the network showed that <i>Collimonas</i>, <i>Roseateles</i>, <i>Mesorhizobium</i>, and <i>Cellulomonas</i> positively correlated with both OM and Po. The random forest showed that <i>Cellulomonas</i>, <i>Roseateles</i>, and <i>Rhodoplanes</i> were the key predictors for ALP activity. The keystone taxon was a more important predictor than the dominant taxon for ALP, OM, and Po. The structural equation model (SEM) showed that soil organic matter (SOM), available P (AP), and OM were the main factors influencing the ALP by reshaping <i>phoD</i>-harboring bacteria alpha diversity, community composition, and <i>phoD</i> abundance.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The <i>phoD</i>-harboring bacterial community composition especially the keystone taxon rather than alpha diversity and abundance dominated the ALP activity, which could promote P utilization over an intensive agroecosystem. These findings improve the understanding of how long-term gradient fertilization influences the community composition and function of P-solubilizing microorganisms in acid purple soil.', attributes={'Label': 'DISCUSSION', 'NlmCategory': 'UNASSIGNED'})]\n",
      "[StringElement('1045919', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1045919', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "<i>phoD</i>-harboring bacterial community composition dominates organic P mineralization under long-term P fertilization in acid purple soil.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Absolute quantification sequencing', attributes={'MajorTopicYN': 'N'}), StringElement('Combination of leguminous shrubs', attributes={'MajorTopicYN': 'N'}), StringElement('Edaphic factor', attributes={'MajorTopicYN': 'N'}), StringElement('Sand-fixing afforestation', attributes={'MajorTopicYN': 'N'}), StringElement('Soil bacterial diversity', attributes={'MajorTopicYN': 'N'}), StringElement('Straw checkerboard', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36495359', attributes={'Version': '1'}), 'DateRevised': {'Year': '2022', 'Month': '12', 'Day': '10'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '12', 'Day': '10'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1007/s00248-022-02151-0', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1432-184X', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'PubDate': {'Year': '2022', 'Month': 'Dec', 'Day': '10'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Microbial ecology', 'ISOAbbreviation': 'Microb Ecol'}, 'ArticleTitle': 'High-Throughput Absolute Quantification Sequencing Reveals that a Combination of Leguminous Shrubs Is Effective in Driving Soil Bacterial Diversity During the Process of Desertification Reversal.', 'Abstract': {'AbstractText': ['Desertification leads to the extreme fragility of ecosystems and seriously threatens ecosystem functioning in desert areas. The planting of xerophytes, especially leguminous shrubs, is an effective and common means to reverse desertification. Soil microorganisms play a crucial role in nutrient cycling and energy flow in ecosystems. However, the effects of introducing leguminous shrubs on soil microbial diversity and the relevant mechanisms are not clear. Here, we employed the high-throughput absolute quantification 16S rRNA sequencing method to analyze the diversity of soil bacteria in sand-fixing areas of mixed shrublands with three combinations of shrubs, i.e., C. korshinskii\\u2009×\\u2009Corethrodendron scoparium (CaKCoS), C. korshinskii\\u2009×\\u2009Calligonum mongolicum (CaKCaM), and C. scoparium\\u2009×\\u2009C. mongolicum (CoSCaM), in the south of the Mu Us Sandy Land, China. This area suffered from moving dunes 20\\xa0years ago, but after introducing these shrubs to fix the dunes, the ecosystem was restored. Additionally, the effects of soil physicochemical properties on soil bacterial composition and diversity were analyzed with redundancy analysis (RDA) and structural equation modeling (SEM). It was found that the Shannon index of soil bacteria in CaKCoS was significantly higher than that in CaKCaM and CoSCaM, and the abundance of the dominant phyla, including Actinobacteria, Proteobacteria, Acidobacteria, Chloroflexi, Planctomycetes, Thaumarchaeota, Armatimonadetes, candidate_division_WPS-1, and Nitrospirae, increased significantly in CaKCoS and CaKCaM compared to that in CoSCaM. RDA showed that the majority of soil properties, such as total nitrogen (TN), available potassium (AK), N:P ratio, soil moisture (SM), and available phosphorus (AP), were important soil environmental factors affecting the abundance of the dominant phyla, and RDA1 and RDA2 accounted for 56.66% and 2.35% of the total variation, respectively. SEM showed that the soil bacterial α-diversity was positively affected by the soil\\xa0organic\\xa0carbon (SOC), N:P ratio, and total phosphorus (TP). Moreover, CaKCoS had higher SM, total carbon (TC), total potassium (TK), and AP than CaKCaM and CoSCaM. Collectively, these results highlight a conceptual framework in which the combination of leguminous shrubs can effectively drive soil bacterial diversity by improving soil physicochemical properties and maintaining ecosystem functioning during desertification reversal.'], 'CopyrightInformation': '© 2022. The Author(s).'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}, {'Identifier': [], 'Affiliation': 'Ningxia Grape Wine and Desertification Prevention Technical College, Yinchuan, 750199, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Liu', 'ForeName': 'Wangsuo', 'Initials': 'W'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China. kaiyangqiu@nxu.edu.cn.'}, {'Identifier': [], 'Affiliation': 'Ningxia Grassland and Animal Husbandry Engineering Technology Research Center, Yinchuan, 750021, Ningxia, China. kaiyangqiu@nxu.edu.cn.'}, {'Identifier': [], 'Affiliation': 'Breeding Base for State Key Laboratory of Land Degradation and Ecological Restoration of Northwest China, Yinchuan, 750021, Ningxia, China. kaiyangqiu@nxu.edu.cn.'}], 'Identifier': [], 'LastName': 'Qiu', 'ForeName': 'Kaiyang', 'Initials': 'K'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}, {'Identifier': [], 'Affiliation': 'Ningxia Grassland and Animal Husbandry Engineering Technology Research Center, Yinchuan, 750021, Ningxia, China.'}, {'Identifier': [], 'Affiliation': 'Breeding Base for State Key Laboratory of Land Degradation and Ecological Restoration of Northwest China, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Xie', 'ForeName': 'Yingzhong', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Huang', 'ForeName': 'Yeyun', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Ningxia Administration of Baijitan National Nature Reserve, Lingwu, 750400, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Ruixia', 'Initials': 'R'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Ningxia Administration of Baijitan National Nature Reserve, Lingwu, 750400, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Haichao', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Meng', 'ForeName': 'Wenfen', 'Initials': 'W'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'He', 'ForeName': 'Yi', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Yayuan', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Li', 'ForeName': 'Haiquan', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Zhao', 'ForeName': 'Pengbo', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'School of Agriculture, Ningxia University, Yinchuan, 750021, Ningxia, China.'}], 'Identifier': [], 'LastName': 'Yang', 'ForeName': 'Yi', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'United States', 'MedlineTA': 'Microb Ecol', 'NlmUniqueID': '7500663', 'ISSNLinking': '0095-3628'}}, attributes={'Status': 'Publisher', 'Owner': 'NLM'})\n",
      "['Desertification leads to the extreme fragility of ecosystems and seriously threatens ecosystem functioning in desert areas. The planting of xerophytes, especially leguminous shrubs, is an effective and common means to reverse desertification. Soil microorganisms play a crucial role in nutrient cycling and energy flow in ecosystems. However, the effects of introducing leguminous shrubs on soil microbial diversity and the relevant mechanisms are not clear. Here, we employed the high-throughput absolute quantification 16S rRNA sequencing method to analyze the diversity of soil bacteria in sand-fixing areas of mixed shrublands with three combinations of shrubs, i.e., C. korshinskii\\u2009×\\u2009Corethrodendron scoparium (CaKCoS), C. korshinskii\\u2009×\\u2009Calligonum mongolicum (CaKCaM), and C. scoparium\\u2009×\\u2009C. mongolicum (CoSCaM), in the south of the Mu Us Sandy Land, China. This area suffered from moving dunes 20\\xa0years ago, but after introducing these shrubs to fix the dunes, the ecosystem was restored. Additionally, the effects of soil physicochemical properties on soil bacterial composition and diversity were analyzed with redundancy analysis (RDA) and structural equation modeling (SEM). It was found that the Shannon index of soil bacteria in CaKCoS was significantly higher than that in CaKCaM and CoSCaM, and the abundance of the dominant phyla, including Actinobacteria, Proteobacteria, Acidobacteria, Chloroflexi, Planctomycetes, Thaumarchaeota, Armatimonadetes, candidate_division_WPS-1, and Nitrospirae, increased significantly in CaKCoS and CaKCaM compared to that in CoSCaM. RDA showed that the majority of soil properties, such as total nitrogen (TN), available potassium (AK), N:P ratio, soil moisture (SM), and available phosphorus (AP), were important soil environmental factors affecting the abundance of the dominant phyla, and RDA1 and RDA2 accounted for 56.66% and 2.35% of the total variation, respectively. SEM showed that the soil bacterial α-diversity was positively affected by the soil\\xa0organic\\xa0carbon (SOC), N:P ratio, and total phosphorus (TP). Moreover, CaKCoS had higher SM, total carbon (TC), total potassium (TK), and AP than CaKCaM and CoSCaM. Collectively, these results highlight a conceptual framework in which the combination of leguminous shrubs can effectively drive soil bacterial diversity by improving soil physicochemical properties and maintaining ecosystem functioning during desertification reversal.']\n",
      "[StringElement('10.1007/s00248-022-02151-0', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "High-Throughput Absolute Quantification Sequencing Reveals that a Combination of Leguminous Shrubs Is Effective in Driving Soil Bacterial Diversity During the Process of Desertification Reversal.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36448819', attributes={'Version': '1'}), 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '02'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '11', 'Day': '30'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('e00831-22', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1128/mra.00831-22', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('2576-098X', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '12', 'Issue': '1', 'PubDate': {'Year': '2023', 'Month': 'Jan', 'Day': '24'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Microbiology resource announcements', 'ISOAbbreviation': 'Microbiol Resour Announc'}, 'ArticleTitle': 'Draft Genome Sequence of a Xylella fastidiosa Strain Causing Bacterial Leaf Scorch of American Elm in Washington, DC.', 'Pagination': {'StartPage': 'e0083122', 'MedlinePgn': 'e0083122'}, 'Abstract': {'AbstractText': ['Here, we report the draft genome sequence of Xylella fastidiosa strain ATCC 35873, which was obtained from the American Type Culture Collection and was originally isolated from a symptomatic American elm tree grown in Washington, DC. The ATCC 35873 genome contains 2,454,216\\u2009bp and has a GC content of 51.68%.']}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Floral and Nursery Plants Research Unit, U.S. National Arboretum, Agricultural Research Service, U.S. Department of Agriculture, Beltsville, Maryland, USA.'}, {'Identifier': [], 'Affiliation': 'State Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China.'}], 'Identifier': [], 'LastName': 'Guan', 'ForeName': 'Wei', 'Initials': 'W'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'North East Area, Agricultural Research Service, U.S. Department of Agriculture, Beltsville, Maryland, USA.'}], 'Identifier': [], 'LastName': 'Shao', 'ForeName': 'Jonathan', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing, China.'}], 'Identifier': [], 'LastName': 'Zhao', 'ForeName': 'Tingchang', 'Initials': 'T'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Floral and Nursery Plants Research Unit, U.S. National Arboretum, Agricultural Research Service, U.S. Department of Agriculture, Beltsville, Maryland, USA.'}], 'Identifier': [StringElement('0000-0003-4278-4491', attributes={'Source': 'ORCID'})], 'LastName': 'Huang', 'ForeName': 'Qi', 'Initials': 'Q'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'United States', 'MedlineTA': 'Microbiol Resour Announc', 'NlmUniqueID': '101728794', 'ISSNLinking': '2576-098X'}, 'CoiStatement': 'The authors declare no conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "['Here, we report the draft genome sequence of Xylella fastidiosa strain ATCC 35873, which was obtained from the American Type Culture Collection and was originally isolated from a symptomatic American elm tree grown in Washington, DC. The ATCC 35873 genome contains 2,454,216\\u2009bp and has a GC content of 51.68%.']\n",
      "[StringElement('e00831-22', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.1128/mra.00831-22', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Draft Genome Sequence of a Xylella fastidiosa Strain Causing Bacterial Leaf Scorch of American Elm in Washington, DC.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36434281', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2023', 'Month': '01', 'Day': '24'}, 'DateRevised': {'Year': '2023', 'Month': '02', 'Day': '01'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '11', 'Day': '24'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('10.1038/s41396-022-01344-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1751-7370', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '17', 'Issue': '2', 'PubDate': {'Year': '2023', 'Month': 'Feb'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'The ISME journal', 'ISOAbbreviation': 'ISME J'}, 'ArticleTitle': 'Intracellular common gardens reveal niche differentiation in transposable element community during bacterial adaptive evolution.', 'Pagination': {'StartPage': '297', 'EndPage': '308', 'MedlinePgn': '297-308'}, 'Abstract': {'AbstractText': ['The distribution and abundance of transposable elements across the tree of life have significantly shaped the evolution of cellular organisms, but the underlying mechanisms shaping these ecological patterns remain elusive. Here we establish a \"common garden\" approach to study causal ecological interactions between a xenogeneic conditional lethal sacB gene and the community of transposable insertion sequences (ISs) in a multipartite prokaryote genome. Xenogeneic sacB of low, medium, or high GC content was individually inserted into three replicons of a model bacterium Sinorhizobium fredii, and exhibited replicon- and GC-dependent variation in genetic stability. This variation was largely attributable to multidimensional niche differentiation for IS community members. The transposition efficiency of major active ISs depended on the nucleoid-associated xenogeneic silencer MucR. Experimentally eliminating insertion activity of specific ISs by deleting MucR strongly demonstrated a dominant role of niche differentiation among ISs. This intracellular common garden approach in the experimental evolution context allows not only for evaluating genetic stability of natural and synthetic xenogeneic genes of different sequence signatures in host cells but also for tracking and testing causal relationships in unifying ecological principles in genome ecology.'], 'CopyrightInformation': '© 2022. The Author(s).'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China.'}], 'Identifier': [], 'LastName': 'Guo', 'ForeName': 'Hui', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China.'}], 'Identifier': [], 'LastName': 'Shi', 'ForeName': 'Wen-Tao', 'Initials': 'WT'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China.'}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Biliang', 'Initials': 'B'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China.'}], 'Identifier': [], 'LastName': 'Xu', 'ForeName': 'Yan-Hui', 'Initials': 'YH'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China.'}], 'Identifier': [StringElement('0000-0002-7889-5823', attributes={'Source': 'ORCID'})], 'LastName': 'Jiao', 'ForeName': 'Jian', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'State Key Laboratory of Agrobiotechnology, and College of Biological Sciences, China Agricultural University, Beijing, China. cftian@cau.edu.cn.'}, {'Identifier': [], 'Affiliation': 'MOA Key Laboratory of Soil Microbiology, and Rhizobium Research Center, China Agricultural University, Beijing, China. cftian@cau.edu.cn.'}], 'Identifier': [StringElement('0000-0002-0479-363X', attributes={'Source': 'ORCID'})], 'LastName': 'Tian', 'ForeName': 'Chang-Fu', 'Initials': 'CF'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'}), 'MedlineJournalInfo': {'Country': 'England', 'MedlineTA': 'ISME J', 'NlmUniqueID': '101301086', 'ISSNLinking': '1751-7362'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('DNA Transposable Elements', attributes={'UI': 'D004251'})}], 'MeshHeadingList': [{'QualifierName': [], 'DescriptorName': StringElement('DNA Transposable Elements', attributes={'UI': 'D004251', 'MajorTopicYN': 'Y'})}, {'QualifierName': [], 'DescriptorName': StringElement('Genome, Bacterial', attributes={'UI': 'D016680', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Prokaryotic Cells', attributes={'UI': 'D011387', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Replicon', attributes={'UI': 'D012093', 'MajorTopicYN': 'N'})}], 'CoiStatement': 'The authors declare no competing interests.'}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "['The distribution and abundance of transposable elements across the tree of life have significantly shaped the evolution of cellular organisms, but the underlying mechanisms shaping these ecological patterns remain elusive. Here we establish a \"common garden\" approach to study causal ecological interactions between a xenogeneic conditional lethal sacB gene and the community of transposable insertion sequences (ISs) in a multipartite prokaryote genome. Xenogeneic sacB of low, medium, or high GC content was individually inserted into three replicons of a model bacterium Sinorhizobium fredii, and exhibited replicon- and GC-dependent variation in genetic stability. This variation was largely attributable to multidimensional niche differentiation for IS community members. The transposition efficiency of major active ISs depended on the nucleoid-associated xenogeneic silencer MucR. Experimentally eliminating insertion activity of specific ISs by deleting MucR strongly demonstrated a dominant role of niche differentiation among ISs. This intracellular common garden approach in the experimental evolution context allows not only for evaluating genetic stability of natural and synthetic xenogeneic genes of different sequence signatures in host cells but also for tracking and testing causal relationships in unifying ecological principles in genome ecology.']\n",
      "[StringElement('10.1038/s41396-022-01344-2', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "Intracellular common gardens reveal niche differentiation in transposable element community during bacterial adaptive evolution.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Geminicoccus flavidas', attributes={'MajorTopicYN': 'N'}), StringElement('Geminicoccus harenae', attributes={'MajorTopicYN': 'N'}), StringElement('average nucleotide identity', attributes={'MajorTopicYN': 'N'}), StringElement('biological soil crusts', attributes={'MajorTopicYN': 'N'}), StringElement('pan-genome', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': [], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36386637', attributes={'Version': '1'}), 'DateRevised': {'Year': '2022', 'Month': '11', 'Day': '18'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [DictElement({'Year': '2022', 'Month': '10', 'Day': '31'}, attributes={'DateType': 'Electronic'})], 'ELocationID': [StringElement('1034816', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1034816', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1664-302X', attributes={'IssnType': 'Print'}), 'JournalIssue': DictElement({'Volume': '13', 'PubDate': {'Year': '2022'}}, attributes={'CitedMedium': 'Print'}), 'Title': 'Frontiers in microbiology', 'ISOAbbreviation': 'Front Microbiol'}, 'ArticleTitle': '<i>Geminicoccus flavidas</i> sp. nov. and <i>Geminicoccus harenae</i> sp. nov., two IAA-producing novel rare bacterial species inhabiting desert biological soil crusts.', 'Pagination': {'StartPage': '1034816', 'MedlinePgn': '1034816'}, 'Abstract': {'AbstractText': ['Two Gram-staining negative strains (CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup>) were isolated from biological sandy soil crusts samples collected from Badain Jaran desert, China. Both isolates were heterotrophic phototroph, could produce indole-3-acetic acid. The 16S rRNA gene sequences of these two strains were closely related to the members of the family <i>Geminicoccaceae</i>, showing high similarities with <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup> (96.9%) and <i>Arboricoccus pini</i> B29T1<sup>T</sup> (90.1%), respectively. In phylogenetic tree based on 16S rRNA gene sequences, strain CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> formed a robust distinct clade with <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup> within the family <i>Geminicoccaceae</i>, which indicated that these two isolates could be classified into the genus <i>Geminicoccus</i>. The growth of strain CPCC 101082<sup>T</sup> occurred at 15-42°C and pH 4.0-10.0 (optima at 28-37°C and pH 6.0-8.0). The growth of strain CPCC 101083<sup>T</sup> occurred at 4-45°C and pH 4.0-10.0 (optima at 25-30°C and pH 6.0-8.0). The major cellular fatty acids of CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> contained C<sub>18:1</sub> <i>ω</i>7<i>c</i>/C<sub>18:1</sub> <i>ω</i>6<i>c</i>, cyclo-C<sub>19:0</sub> <i>ω</i>8<i>c</i>, and C<sub>16:0</sub>. Q-10 was detected as the sole respiratory quinone. Diphosphatidylglycerol, phosphatidylglycerol, phosphatidylcholine, phosphatidylethanolamine, an unidentified phospholipid and an unidentified aminolipid were tested in the polar lipids profile. The genomes of the two isolates were characterized as about 5.9 Mbp in size with the G\\u2009+\\u2009C content of nearly 68%. The IAA-producing encoding genes were predicated in both genomes. The values of average nucleotide identity were 80.6, 81.2 and 92.4% based on a pairwise comparison of the genomes of strains CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> and <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup>, respectively. On the basis of the genotypic, chemotaxonomic and phenotypic characteristics, the strains CPCC 101082<sup>T</sup> (=NBRC 113513<sup>T</sup>\\u2009=\\u2009KCTC 62853<sup>T</sup>) and CPCC 101083<sup>T</sup> (=NBRC 113514<sup>T</sup>\\u2009=\\u2009KCTC 62854<sup>T</sup>) are proposed to represent two novel species of the genus <i>Geminicoccus</i> with the names <i>Geminicoccus flavidas</i> sp. nov. and <i>Geminicoccus harenae</i> sp. nov.'], 'CopyrightInformation': 'Copyright © 2022 Jiang, Deng, Han, Su, Wang, Yu and Zhang.'}, 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'State Key Laboratory of Dao-di Herb, Beijing, China.'}], 'Identifier': [], 'LastName': 'Jiang', 'ForeName': 'Zhu-Ming', 'Initials': 'ZM'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'State Key Laboratory of Dao-di Herb, Beijing, China.'}], 'Identifier': [], 'LastName': 'Deng', 'ForeName': 'Yang', 'Initials': 'Y'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'State Key Laboratory of Dao-di Herb, Beijing, China.'}], 'Identifier': [], 'LastName': 'Han', 'ForeName': 'Xue-Fei', 'Initials': 'XF'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}], 'Identifier': [], 'LastName': 'Su', 'ForeName': 'Jing', 'Initials': 'J'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}], 'Identifier': [], 'LastName': 'Wang', 'ForeName': 'Hao', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}], 'Identifier': [], 'LastName': 'Yu', 'ForeName': 'Li-Yan', 'Initials': 'LY'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China.'}, {'Identifier': [], 'Affiliation': 'State Key Laboratory of Dao-di Herb, Beijing, China.'}], 'Identifier': [], 'LastName': 'Zhang', 'ForeName': 'Yu-Qin', 'Initials': 'YQ'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Electronic-eCollection'}), 'MedlineJournalInfo': {'Country': 'Switzerland', 'MedlineTA': 'Front Microbiol', 'NlmUniqueID': '101548977', 'ISSNLinking': '1664-302X'}, 'CoiStatement': 'The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.'}, attributes={'Status': 'PubMed-not-MEDLINE', 'Owner': 'NLM'})\n",
      "['Two Gram-staining negative strains (CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup>) were isolated from biological sandy soil crusts samples collected from Badain Jaran desert, China. Both isolates were heterotrophic phototroph, could produce indole-3-acetic acid. The 16S rRNA gene sequences of these two strains were closely related to the members of the family <i>Geminicoccaceae</i>, showing high similarities with <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup> (96.9%) and <i>Arboricoccus pini</i> B29T1<sup>T</sup> (90.1%), respectively. In phylogenetic tree based on 16S rRNA gene sequences, strain CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> formed a robust distinct clade with <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup> within the family <i>Geminicoccaceae</i>, which indicated that these two isolates could be classified into the genus <i>Geminicoccus</i>. The growth of strain CPCC 101082<sup>T</sup> occurred at 15-42°C and pH 4.0-10.0 (optima at 28-37°C and pH 6.0-8.0). The growth of strain CPCC 101083<sup>T</sup> occurred at 4-45°C and pH 4.0-10.0 (optima at 25-30°C and pH 6.0-8.0). The major cellular fatty acids of CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> contained C<sub>18:1</sub> <i>ω</i>7<i>c</i>/C<sub>18:1</sub> <i>ω</i>6<i>c</i>, cyclo-C<sub>19:0</sub> <i>ω</i>8<i>c</i>, and C<sub>16:0</sub>. Q-10 was detected as the sole respiratory quinone. Diphosphatidylglycerol, phosphatidylglycerol, phosphatidylcholine, phosphatidylethanolamine, an unidentified phospholipid and an unidentified aminolipid were tested in the polar lipids profile. The genomes of the two isolates were characterized as about 5.9 Mbp in size with the G\\u2009+\\u2009C content of nearly 68%. The IAA-producing encoding genes were predicated in both genomes. The values of average nucleotide identity were 80.6, 81.2 and 92.4% based on a pairwise comparison of the genomes of strains CPCC 101082<sup>T</sup> and CPCC 101083<sup>T</sup> and <i>Geminicoccus roseus</i> DSM 18922<sup>T</sup>, respectively. On the basis of the genotypic, chemotaxonomic and phenotypic characteristics, the strains CPCC 101082<sup>T</sup> (=NBRC 113513<sup>T</sup>\\u2009=\\u2009KCTC 62853<sup>T</sup>) and CPCC 101083<sup>T</sup> (=NBRC 113514<sup>T</sup>\\u2009=\\u2009KCTC 62854<sup>T</sup>) are proposed to represent two novel species of the genus <i>Geminicoccus</i> with the names <i>Geminicoccus flavidas</i> sp. nov. and <i>Geminicoccus harenae</i> sp. nov.']\n",
      "[StringElement('1034816', attributes={'EIdType': 'pii', 'ValidYN': 'Y'}), StringElement('10.3389/fmicb.2022.1034816', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "<i>Geminicoccus flavidas</i> sp. nov. and <i>Geminicoccus harenae</i> sp. nov., two IAA-producing novel rare bacterial species inhabiting desert biological soil crusts.\n",
      "DictElement({'SpaceFlightMission': [], 'KeywordList': [ListElement([StringElement('Lactuca sativa var. capitata', attributes={'MajorTopicYN': 'N'}), StringElement('bacterial rot disease', attributes={'MajorTopicYN': 'N'}), StringElement('lettuce', attributes={'MajorTopicYN': 'N'}), StringElement('phylogenomic analysis', attributes={'MajorTopicYN': 'N'}), StringElement('polyphasic taxonomy', attributes={'MajorTopicYN': 'N'})], attributes={'Owner': 'NOTNLM'})], 'GeneralNote': [], 'CitationSubset': ['IM'], 'OtherID': [], 'OtherAbstract': [], 'PMID': StringElement('36331109', attributes={'Version': '1'}), 'DateCompleted': {'Year': '2022', 'Month': '11', 'Day': '07'}, 'DateRevised': {'Year': '2023', 'Month': '03', 'Day': '02'}, 'Article': DictElement({'Language': ['eng'], 'ArticleDate': [], 'ELocationID': [StringElement('10.1099/ijsem.0.005599', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'Journal': {'ISSN': StringElement('1466-5034', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '72', 'Issue': '11', 'PubDate': {'Year': '2022', 'Month': 'Nov'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'International journal of systematic and evolutionary microbiology', 'ISOAbbreviation': 'Int J Syst Evol Microbiol'}, 'ArticleTitle': '<i>Pseudomonas aegrilactucae</i> sp. nov. and <i>Pseudomonas morbosilactucae</i> sp. nov., pathogens causing bacterial rot of lettuce in Japan.', 'AuthorList': ListElement([DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Research Center of Genetic Resources, National Agriculture and Food Research Organization (NARO), 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan.'}], 'Identifier': [], 'LastName': 'Sawada', 'ForeName': 'Hiroyuki', 'Initials': 'H'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute for Plant Protection, NARO, 2-1-18 Kannondai, Tsukuba, Ibaraki 305-8666, Japan.'}], 'Identifier': [], 'LastName': 'Fujikawa', 'ForeName': 'Takashi', 'Initials': 'T'}, attributes={'ValidYN': 'Y'}), DictElement({'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Research Center of Genetic Resources, National Agriculture and Food Research Organization (NARO), 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan.'}], 'Identifier': [], 'LastName': 'Satou', 'ForeName': 'Mamoru', 'Initials': 'M'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print'}), 'MedlineJournalInfo': {'Country': 'England', 'MedlineTA': 'Int J Syst Evol Microbiol', 'NlmUniqueID': '100899600', 'ISSNLinking': '1466-5026'}, 'ChemicalList': [{'RegistryNumber': '0', 'NameOfSubstance': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('DNA, Bacterial', attributes={'UI': 'D004269'})}, {'RegistryNumber': '0', 'NameOfSubstance': StringElement('Fatty Acids', attributes={'UI': 'D005227'})}], 'MeshHeadingList': [{'QualifierName': [StringElement('microbiology', attributes={'UI': 'Q000382', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Lettuce', attributes={'UI': 'D018545', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Pseudomonas', attributes={'UI': 'D011549', 'MajorTopicYN': 'Y'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('RNA, Ribosomal, 16S', attributes={'UI': 'D012336', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('genetics', attributes={'UI': 'Q000235', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('DNA, Bacterial', attributes={'UI': 'D004269', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacterial Typing Techniques', attributes={'UI': 'D015373', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Japan', attributes={'UI': 'D007564', 'MajorTopicYN': 'N', 'Type': 'Geographic'})}, {'QualifierName': [], 'DescriptorName': StringElement('Phylogeny', attributes={'UI': 'D010802', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Sequence Analysis, DNA', attributes={'UI': 'D017422', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Base Composition', attributes={'UI': 'D001482', 'MajorTopicYN': 'N'})}, {'QualifierName': [StringElement('chemistry', attributes={'UI': 'Q000737', 'MajorTopicYN': 'N'})], 'DescriptorName': StringElement('Fatty Acids', attributes={'UI': 'D005227', 'MajorTopicYN': 'N'})}, {'QualifierName': [], 'DescriptorName': StringElement('Bacteria', attributes={'UI': 'D001419', 'MajorTopicYN': 'N'})}]}, attributes={'Status': 'MEDLINE', 'Owner': 'NLM', 'IndexingMethod': 'Automated'})\n",
      "\n",
      "[StringElement('10.1099/ijsem.0.005599', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})]\n",
      "<i>Pseudomonas aegrilactucae</i> sp. nov. and <i>Pseudomonas morbosilactucae</i> sp. nov., pathogens causing bacterial rot of lettuce in Japan.\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "# Search PubMed for articles with \"GC content bacterial\" in the title\n",
    "term = \"GC content bacterial[Title]\"\n",
    "handle = Entrez.esearch(db=\"pubmed\", term=term)\n",
    "record = Entrez.read(handle)\n",
    "ids = record[\"IdList\"]\n",
    "\n",
    "# Retrieve the abstracts for the articles\n",
    "handle = Entrez.efetch(db=\"pubmed\", id=ids, retmode=\"xml\")\n",
    "records = Entrez.read(handle)\n",
    "\n",
    "# Extract relevant fields from each article\n",
    "relevant_fields = []\n",
    "for record in records[\"PubmedArticle\"]:\n",
    "    temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "    doi = temp_record.get(\"ELocationID\", \"\") or temp_record.get(\"doi\", \"\")\n",
    "    title = temp_record.get(\"ArticleTitle\", \"\")\n",
    "    abstract = record[\"MedlineCitation\"].get(\"Article\", {}).get(\"Abstract\", {}).get(\"AbstractText\", \"\")\n",
    "    relevant_fields.append({\"MedlineCitation\": record[\"MedlineCitation\"], \"Abstract\": abstract, \"doi\": doi, \"title\": title})\n",
    "\n",
    "# Print the selected fields for each article\n",
    "for article in relevant_fields:\n",
    "    print(article[\"MedlineCitation\"])\n",
    "    print(article[\"Abstract\"])\n",
    "    print(article[\"doi\"])\n",
    "    print(article[\"title\"])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a15fbc13",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import re\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "# Search PubMed for articles with \"GC content bacterial\" in the title\n",
    "term = \"GC content bacterial[Title]\"\n",
    "handle = Entrez.esearch(db=\"pubmed\", term=term)\n",
    "record = Entrez.read(handle)\n",
    "ids = record[\"IdList\"]\n",
    "\n",
    "# Retrieve the abstracts for the articles\n",
    "handle = Entrez.efetch(db=\"pubmed\", id=ids, rettype=\"abstract\", retmode=\"xml\")\n",
    "\n",
    "records = Entrez.read(handle)\n",
    "\n",
    "# Select articles with \"GC\", \"content\", and \"bacterial\" in the title and 1% in the abstract\n",
    "abstracts_with_one_perc = []\n",
    "for record in records[\"PubmedArticle\"]:\n",
    "    try:\n",
    "        temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "        title = temp_record[\"ArticleTitle\"]\n",
    "        abstract = temp_record[\"Abstract\"][\"AbstractText\"][0]\n",
    "        if \"GC\" in title and \"content\" in title and \"bacterial\" in title and \"%\" in abstract:\n",
    "            matches = re.findall(r\"\\%\", abstract)\n",
    "            if len(matches) == 1:\n",
    "                doi = temp_record.get('ELocationID', None)\n",
    "                if isinstance(doi, list):\n",
    "                    doi = doi[0]\n",
    "                abstracts_with_one_perc.append((temp_record['MedlineCitation'], temp_record['Article'], doi, title))\n",
    "    except:\n",
    "        pass\n",
    "\n",
    "# Print the selected records\n",
    "for record in abstracts_with_one_perc:\n",
    "    medline_citation = record[0]\n",
    "    article = record[1]\n",
    "    doi = record[2]\n",
    "    title = record[3]\n",
    "    \n",
    "    print(\"MedlineCitation:\", medline_citation)\n",
    "    print(\"Abstract:\", article['Abstract']['AbstractText'][0])\n",
    "    print(\"DOI:\", doi)\n",
    "    print(\"Title:\", title)\n",
    "    print(\"--------------------------------------------------------\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "fa24f8a6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available\n",
      "No abstract available for 1 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n"
     ]
    }
   ],
   "source": [
    "import re\n",
    "from Bio import Entrez\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "abstracts_with_one_perc = []\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Print the abstracts\n",
    "    noAbstract = 0\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            matches = re.findall(r\"\\%\", temp_record[\"Abstract\"][\"AbstractText\"][0])\n",
    "            if(len(matches) == 1):\n",
    "                abstracts_with_one_perc.append(temp_record)\n",
    "            pass\n",
    "        except KeyError:\n",
    "            print(\"No abstract available\")\n",
    "            noAbstract += 1\n",
    "    print(\"No abstract available for \" + str(noAbstract) + \" articles \"  + \"by total of \" + str(len(article_ids)) + \" articles\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "eb60478a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available\n",
      "No abstract available for 1 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "abstracts_with_one_perc = []\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Print the abstracts\n",
    "    noAbstract = 0\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            matches = re.findall(r\"\\%\", temp_record[\"Abstract\"][\"AbstractText\"][0])\n",
    "            if(len(matches) == 1):\n",
    "                abstracts_with_one_perc.append(temp_record)\n",
    "            pass\n",
    "        except KeyError:\n",
    "            print(\"No abstract available\")\n",
    "            noAbstract += 1\n",
    "    print(\"No abstract available for \" + str(noAbstract) + \" articles \"  + \"by total of \" + str(len(article_ids)) + \" articles\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "0b94c738",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'The protein expressions of three preferred human codon-optimized Gaussia luciferase genes (pGLuc, EpGLuc, and KpGLuc) were characterized in mammalian and bacterial cells by comparing them with those of wild-type Gaussia luciferase gene (wGLuc) and human codon-optimized Gaussia luciferase gene (hGLuc). Two synthetic genes of EpGLuc and KpGLuc containing the complete preferred human codons have an artificial open-reading frame; however, they had the similar protein expression levels to those of pGLuc and hGLuc in mammalian cells. In bacterial cells, the protein expressions of pGLuc, EpGLuc, and KpGLuc with approximately 65% GC content were the same and showed approximately 60% activities of wGLuc and hGLuc. The artificial open-reading frame in EpGLuc and KpGLuc did not affect the protein expression in mammalian and bacterial cells.'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "records[\"PubmedArticle\"][0][\"MedlineCitation\"][\"Article\"][\"Abstract\"][\"AbstractText\"][0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "cd9b971f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[StringElement('Journal Article', attributes={'UI': 'D016428'})]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "records[\"PubmedArticle\"][0][\"MedlineCitation\"][\"Article\"].keys()\n",
    "records[\"PubmedArticle\"][0][\"MedlineCitation\"][\"Article\"][\"PublicationTypeList\"]\n",
    "# records[\"PubmedArticle\"][0][\"MedlineCitation\"].keys()\n",
    "# records[\"PubmedArticle\"][0][\"MedlineCitation\"].keys()\n",
    "# records[\"PubmedArticle\"][0].keys()\n",
    "\n",
    "\n",
    "# records[\"PubmedArticle\"][0][\"PubmedData\"].keys()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "46e21f1e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[StringElement('10.1016/j.pep.2016.08.006', attributes={'EIdType': 'doi', 'ValidYN': 'Y'}),\n",
       " StringElement('S1046-5928(16)30154-1', attributes={'EIdType': 'pii', 'ValidYN': 'Y'})]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "records[\"PubmedArticle\"][0][\"MedlineCitation\"][\"Article\"][\"ELocationID\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "ee563740",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Is the data for None with 43.88% correct? (y/n/m)\n",
      "Is the data for None with 43.88% correct? (y/n/m)y\n",
      "Is the data for None with 56% correct? (y/n/m)\n",
      "Is the data for None with 56% correct? (y/n/m)y\n",
      "Is the data for None with 53.50% correct? (y/n/m)\n",
      "Is the data for None with 53.50% correct? (y/n/m)y\n",
      "Is the data for None with 51.68% correct? (y/n/m)\n",
      "Is the data for None with 51.68% correct? (y/n/m)y\n",
      "Is the data for Martelella soudanensis with None correct? (y/n/m)\n",
      "Is the data for Martelella soudanensis with None correct? (y/n/m)y\n",
      "Is the data for None with None correct? (y/n/m)\n",
      "Is the data for None with None correct? (y/n/m)y\n",
      "Is the data for E. coli with 7.4% correct? (y/n/m)\n",
      "Is the data for E. coli with 7.4% correct? (y/n/m)y\n",
      "Is the data for Oceanobacillus jordanicus with 39.09% correct? (y/n/m)\n",
      "Is the data for Oceanobacillus jordanicus with 39.09% correct? (y/n/m)y\n",
      "Is the data for Lactobacillus rhamnosus with None correct? (y/n/m)\n",
      "Is the data for Lactobacillus rhamnosus with None correct? (y/n/m)y\n",
      "Is the data for Pediococcus pentosaceus with None correct? (y/n/m)\n",
      "Is the data for Pediococcus pentosaceus with None correct? (y/n/m)y\n",
      "Is the data for Xanthomonas oryzae with 63.7% correct? (y/n/m)\n",
      "Is the data for Xanthomonas oryzae with 63.7% correct? (y/n/m)y\n",
      "Is the data for Bacillus with 41.41% correct? (y/n/m)\n",
      "Is the data for Bacillus with 41.41% correct? (y/n/m)y\n",
      "Is the data for Streptomyces with 71.80% correct? (y/n/m)\n",
      "Is the data for Streptomyces with 71.80% correct? (y/n/m)y\n",
      "Is the data for None with 39.2% correct? (y/n/m)\n",
      "Is the data for None with 39.2% correct? (y/n/m)y\n",
      "Is the data for Methylocystis with None correct? (y/n/m)\n",
      "Is the data for Methylocystis with None correct? (y/n/m)y\n",
      "Is the data for Paenibacillus with 46% correct? (y/n/m)\n",
      "Is the data for Paenibacillus with 46% correct? (y/n/m)y\n",
      "Is the data for Enterococcus faecalis with None correct? (y/n/m)\n",
      "Is the data for Enterococcus faecalis with None correct? (y/n/m)y\n",
      "Is the data for Enterobacter roggenkampii with 56.16% correct? (y/n/m)\n",
      "Is the data for Enterobacter roggenkampii with 56.16% correct? (y/n/m)y\n",
      "Is the data for None with None correct? (y/n/m)\n",
      "Is the data for None with None correct? (y/n/m)y\n",
      "Is the data for None with 39% correct? (y/n/m)\n",
      "Is the data for None with 39% correct? (y/n/m)y\n",
      "Is the data for None with 0.9% correct? (y/n/m)\n",
      "Is the data for None with 0.9% correct? (y/n/m)y\n",
      "Is the data for None with 27.6% correct? (y/n/m)\n",
      "Is the data for None with 27.6% correct? (y/n/m)y\n",
      "Is the data for None with 53.8% correct? (y/n/m)\n",
      "Is the data for None with 53.8% correct? (y/n/m)y\n",
      "Is the data for Pseudarthrobacter phenanthrenivorans with 65.30% correct? (y/n/m)\n",
      "Is the data for Pseudarthrobacter phenanthrenivorans with 65.30% correct? (y/n/m)y\n",
      "Is the data for None with 64% correct? (y/n/m)\n",
      "Is the data for None with 64% correct? (y/n/m)y\n",
      "Is the data for None with None correct? (y/n/m)\n",
      "Is the data for None with None correct? (y/n/m)y\n",
      "Is the data for Methanococcus maripaludis with 94% correct? (y/n/m)\n",
      "Is the data for Methanococcus maripaludis with 94% correct? (y/n/m)y\n",
      "Is the data for None with 59.2% correct? (y/n/m)\n",
      "Is the data for None with 59.2% correct? (y/n/m)y\n",
      "Is the data for 'Cellulomonas timonensis' with 72.42% correct? (y/n/m)\n",
      "Is the data for 'Cellulomonas timonensis' with 72.42% correct? (y/n/m)y\n",
      "Is the data for Acidovorax citrulli with 58.7% correct? (y/n/m)\n",
      "Is the data for Acidovorax citrulli with 58.7% correct? (y/n/m)y\n",
      "Is the data for None with 71% correct? (y/n/m)\n",
      "Is the data for None with 71% correct? (y/n/m)y\n",
      "Is the data for None with 200% correct? (y/n/m)\n",
      "Is the data for None with 200% correct? (y/n/m)y\n",
      "Is the data for Methylobacterium radiotolerans with 70.5% correct? (y/n/m)\n",
      "Is the data for Methylobacterium radiotolerans with 70.5% correct? (y/n/m)y\n",
      "Is the data for Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella with 75% correct? (y/n/m)\n",
      "Is the data for Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella with 75% correct? (y/n/m)y\n",
      "Is the data for Lactobacillus murinus with 39.6% correct? (y/n/m)\n",
      "Is the data for Lactobacillus murinus with 39.6% correct? (y/n/m)y\n",
      "Is the data for None with 68.66% correct? (y/n/m)\n",
      "Is the data for None with 68.66% correct? (y/n/m)y\n",
      "Is the data for None with 95% correct? (y/n/m)\n",
      "Is the data for None with 95% correct? (y/n/m)y\n",
      "Is the data for None with None correct? (y/n/m)\n",
      "Is the data for None with None correct? (y/n/m)y\n",
      "Is the data for None with 34.4% correct? (y/n/m)\n",
      "Is the data for None with 34.4% correct? (y/n/m)y\n",
      "Is the data for None with None correct? (y/n/m)\n",
      "Is the data for None with None correct? (y/n/m)y\n",
      "Is the data for None with 67.81% correct? (y/n/m)\n",
      "Is the data for None with 67.81% correct? (y/n/m)y\n",
      "Is the data for None with 44.8% correct? (y/n/m)\n",
      "Is the data for None with 44.8% correct? (y/n/m)y\n",
      "Is the data for None with 52.46% correct? (y/n/m)\n",
      "Is the data for None with 52.46% correct? (y/n/m)y\n",
      "Is the data for None with 2% correct? (y/n/m)\n",
      "Is the data for None with 2% correct? (y/n/m)y\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[{None: '43.88%', 'is_valuable_data': 'y'},\n",
       " {None: '56%', 'is_valuable_data': 'y'},\n",
       " {None: '53.50%', 'is_valuable_data': 'y'},\n",
       " {None: '51.68%', 'is_valuable_data': 'y'},\n",
       " {'Martelella soudanensis': None, 'is_valuable_data': 'y'},\n",
       " {None: None, 'is_valuable_data': 'y'},\n",
       " {'E. coli': '7.4%', 'is_valuable_data': 'y'},\n",
       " {'Oceanobacillus jordanicus': '39.09%', 'is_valuable_data': 'y'},\n",
       " {'Lactobacillus rhamnosus': None, 'is_valuable_data': 'y'},\n",
       " {'Pediococcus pentosaceus': None, 'is_valuable_data': 'y'},\n",
       " {'Xanthomonas oryzae': '63.7%', 'is_valuable_data': 'y'},\n",
       " {'Bacillus': '41.41%', 'is_valuable_data': 'y'},\n",
       " {'Streptomyces': '71.80%', 'is_valuable_data': 'y'},\n",
       " {None: '39.2%', 'is_valuable_data': 'y'},\n",
       " {'Methylocystis': None, 'is_valuable_data': 'y'},\n",
       " {'Paenibacillus': '46%', 'is_valuable_data': 'y'},\n",
       " {'Enterococcus faecalis': None, 'is_valuable_data': 'y'},\n",
       " {'Enterobacter roggenkampii': '56.16%', 'is_valuable_data': 'y'},\n",
       " {None: None, 'is_valuable_data': 'y'},\n",
       " {None: '39%', 'is_valuable_data': 'y'},\n",
       " {None: '0.9%', 'is_valuable_data': 'y'},\n",
       " {None: '27.6%', 'is_valuable_data': 'y'},\n",
       " {None: '53.8%', 'is_valuable_data': 'y'},\n",
       " {'Pseudarthrobacter phenanthrenivorans': '65.30%', 'is_valuable_data': 'y'},\n",
       " {None: '64%', 'is_valuable_data': 'y'},\n",
       " {None: None, 'is_valuable_data': 'y'},\n",
       " {'Methanococcus maripaludis': '94%', 'is_valuable_data': 'y'},\n",
       " {None: '59.2%', 'is_valuable_data': 'y'},\n",
       " {\"'Cellulomonas timonensis'\": '72.42%', 'is_valuable_data': 'y'},\n",
       " {'Acidovorax citrulli': '58.7%', 'is_valuable_data': 'y'},\n",
       " {None: '71%', 'is_valuable_data': 'y'},\n",
       " {None: '200%', 'is_valuable_data': 'y'},\n",
       " {'Methylobacterium radiotolerans': '70.5%', 'is_valuable_data': 'y'},\n",
       " {'Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella': '75%',\n",
       "  'is_valuable_data': 'y'},\n",
       " {'Lactobacillus murinus': '39.6%', 'is_valuable_data': 'y'},\n",
       " {None: '68.66%', 'is_valuable_data': 'y'},\n",
       " {None: '95%', 'is_valuable_data': 'y'},\n",
       " {None: None, 'is_valuable_data': 'y'},\n",
       " {None: '34.4%', 'is_valuable_data': 'y'},\n",
       " {None: None, 'is_valuable_data': 'y'},\n",
       " {None: '67.81%', 'is_valuable_data': 'y'},\n",
       " {None: '44.8%', 'is_valuable_data': 'y'},\n",
       " {None: '52.46%', 'is_valuable_data': 'y'},\n",
       " {None: '2%', 'is_valuable_data': 'y'}]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "hopefull_dicts = []\n",
    "\n",
    "# give the one <i> tag which is the nearest to the % sign\n",
    "\n",
    "# Add DOI to the dict\n",
    "# Add Abstract to the dict\n",
    "# ETC\n",
    "\n",
    "\n",
    "for pre_x in abstracts_with_one_perc:\n",
    "    x = pre_x[\"Abstract\"][\"AbstractText\"][0]\n",
    "    \n",
    "    # Extract the content between the tags\n",
    "    result = re.search(r\"<i>(.*?)</i>\", x)\n",
    "    if result:\n",
    "        hopefully_bacterial = result.group(1)\n",
    "    else:\n",
    "        hopefully_bacterial = None\n",
    "\n",
    "\n",
    "    # Extract the precentage value\n",
    "    result = re.search(r\"([\\d\\.]+\\%)\", x)\n",
    "    if result:\n",
    "        hopefully_percentage_for_bacterial = result.group(1)\n",
    "    else:\n",
    "        hopefully_percentage_for_bacterial = None\n",
    "\n",
    "    is_valuable_data = \"\"\n",
    "    print(\"Is the data for \" + str(hopefully_bacterial) + \" with \" + str(hopefully_percentage_for_bacterial) + \" correct? (y/n/m)\")\n",
    "    while(is_valuable_data != \"y\" and is_valuable_data != \"n\" and is_valuable_data != \"m\"):\n",
    "        is_valuable_data = input(\"Is the data for \" + str(hopefully_bacterial) + \" with \" + str(hopefully_percentage_for_bacterial) + \" correct? (y/n/m)\")\n",
    "\n",
    "    hopefull_dict = {hopefully_bacterial: hopefully_percentage_for_bacterial, \"is_valuable_data\": is_valuable_data}\n",
    "\n",
    "    hopefull_dicts.append(hopefull_dict)\n",
    "\n",
    "# show only dicts with a value and a key\n",
    "# hopefull_dicts = [x for x in hopefull_dicts if list(x.keys())[0] is not None and list(x.values())[0] is not None]\n",
    "hopefull_dicts\n",
    "# how do we verify?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "6a1456a7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# only show the ones which are valuable\n",
    "[x for x in hopefull_dicts if x[\"is_valuable_data\"] == \"m\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "d0c42dc3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unknown has a GC content of 43.88%\n",
      "Unknown has a GC content of 56%\n",
      "Unknown has a GC content of 53.50%\n",
      "Unknown has a GC content of 51.68%\n",
      "Martelella soudanensis has a GC content of Unknown\n",
      "Unknown has a GC content of Unknown\n",
      "E. coli has a GC content of 7.4%\n",
      "Oceanobacillus jordanicus has a GC content of 39.09%\n",
      "Lactobacillus rhamnosus has a GC content of Unknown\n",
      "Pediococcus pentosaceus has a GC content of Unknown\n",
      "Xanthomonas oryzae has a GC content of 63.7%\n",
      "Bacillus has a GC content of 41.41%\n",
      "Streptomyces has a GC content of 71.80%\n",
      "Unknown has a GC content of 39.2%\n",
      "Methylocystis has a GC content of Unknown\n",
      "Paenibacillus has a GC content of 46%\n",
      "Enterococcus faecalis has a GC content of Unknown\n",
      "Enterobacter roggenkampii has a GC content of 56.16%\n",
      "Unknown has a GC content of Unknown\n",
      "Unknown has a GC content of 39%\n",
      "Unknown has a GC content of 0.9%\n",
      "Unknown has a GC content of 27.6%\n",
      "Unknown has a GC content of 53.8%\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30%\n",
      "Unknown has a GC content of 64%\n",
      "Unknown has a GC content of Unknown\n",
      "Methanococcus maripaludis has a GC content of 94%\n",
      "Unknown has a GC content of 59.2%\n",
      "'Cellulomonas timonensis' has a GC content of 72.42%\n",
      "Acidovorax citrulli has a GC content of 58.7%\n",
      "Unknown has a GC content of 71%\n",
      "Unknown has a GC content of 200%\n",
      "Methylobacterium radiotolerans has a GC content of 70.5%\n",
      "Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella has a GC content of 75%\n",
      "Lactobacillus murinus has a GC content of 39.6%\n",
      "Unknown has a GC content of 68.66%\n",
      "Unknown has a GC content of 95%\n",
      "Unknown has a GC content of Unknown\n",
      "Unknown has a GC content of 34.4%\n",
      "Unknown has a GC content of Unknown\n",
      "Unknown has a GC content of 67.81%\n",
      "Unknown has a GC content of 44.8%\n",
      "Unknown has a GC content of 52.46%\n",
      "Unknown has a GC content of 2%\n"
     ]
    }
   ],
   "source": [
    "for abstract in abstracts_with_one_perc:\n",
    "    text = abstract[\"Abstract\"][\"AbstractText\"][0]\n",
    "    bacterial_match = re.search(r\"<i>(.*?)</i>\", text)\n",
    "    if bacterial_match:\n",
    "        bacterial_name = bacterial_match.group(1)\n",
    "    else:\n",
    "        bacterial_name = \"Unknown\"\n",
    "    gc_match = re.search(r\"([\\d\\.]+\\%)\", text)\n",
    "    if gc_match:\n",
    "        gc_content = gc_match.group(1)\n",
    "    else:\n",
    "        gc_content = \"Unknown\"\n",
    "    print(f\"{bacterial_name} has a GC content of {gc_content}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "ba14ca81",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No abstract available\n",
      "Xanthomonas oryzae has a GC content of 80%\n",
      "Geminicoccaceae has a GC content of 96.9%\n",
      "Triticum aestivum has a GC content of 5%\n",
      "Petroselinum crispum has a GC content of 99.93%\n",
      "Brachybacterium muris has a GC content of 99.5%\n",
      "E. coli has a GC content of 7.4%\n",
      "Peptostreptococcus anaerobius has a GC content of 96.2%\n",
      "Oceanobacillus jordanicus has a GC content of 39.09%\n",
      "Acidobacteria has a GC content of 29.2%\n",
      "Thymus vulgaris has a GC content of 100%\n",
      "Oe has a GC content of 93.3%\n",
      "Xanthomonas oryzae has a GC content of 63.7%\n",
      "Bacillus has a GC content of 41.41%\n",
      ". Neobacillus massiliamazoniensis has a GC content of 98.32%\n",
      "Ralstonia solanacearum has a GC content of 68%\n",
      "Actinomycetales has a GC content of 94.73%\n",
      "Schleiferiaceae has a GC content of 46.6%\n",
      "Streptomyces has a GC content of 71.80%\n",
      "Paenibacillus has a GC content of 46%\n",
      "Enterobacter roggenkampii has a GC content of 56.16%\n",
      "Erysipelothrix rhusiopathiae has a GC content of 36.49%\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30%\n",
      "Allium fistulosum has a GC content of 99.72%\n",
      "n has a GC content of 5%\n",
      "Acidobacteria has a GC content of 57%\n",
      "Bacillus has a GC content of 35.23%\n",
      "Dielma fastidiosa has a GC content of 93.2%\n",
      "Bacillus velezensis has a GC content of 46.5%\n",
      "Agaricus bisporus. has a GC content of 55%\n",
      "Alteromonadales bacterium has a GC content of 53%\n",
      "Methanococcus maripaludis has a GC content of 94%\n",
      "Mauritia flexuosa has a GC content of 89.81%\n",
      "Xanthomonas euvesicatoria has a GC content of 62.9%\n",
      "'Cellulomonas timonensis' has a GC content of 72.42%\n",
      "Acidovorax citrulli has a GC content of 58.7%\n",
      "Methylobacterium radiotolerans has a GC content of 70.5%\n",
      "Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella has a GC content of 75%\n",
      "Lactobacillus murinus has a GC content of 39.6%\n",
      "Rubeoparvulum massiliense has a GC content of 92%\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import re\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "abstracts_with_gc = []\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Parse the abstracts\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            abstract = temp_record[\"Abstract\"][\"AbstractText\"][0]\n",
    "            gc_match = re.search(r\"([\\d\\.]+\\%)\", abstract)\n",
    "            bacterial_match = re.search(r\"<i>(.*?)</i>\", abstract)\n",
    "            if gc_match and bacterial_match:\n",
    "                gc_content = gc_match.group(1)\n",
    "                bacterial_name = bacterial_match.group(1)\n",
    "                abstracts_with_gc.append((bacterial_name, gc_content))\n",
    "        except KeyError:\n",
    "            print(\"No abstract available\")\n",
    "\n",
    "# Print the bacterial names and their corresponding GC content percentages\n",
    "for bacterial_name, gc_content in abstracts_with_gc:\n",
    "    print(f\"{bacterial_name} has a GC content of {gc_content}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e7ec6391",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Xanthomonas  Hat eine große Range an GC content\n",
    "#Reicht es, wenn wir einen konkreten Wert aus einer Studie ziehen, oder müssen wir die Range ermitteln?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a71e7a65",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available\n",
      "No abstract available for 1 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "No abstract available for 0 articles by total of 20 articles\n",
      "Unknown has a GC content of 43.88%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 56%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 53.50%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 51.68%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Martelella soudanensis has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "E. coli has a GC content of 7.4%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Oceanobacillus jordanicus has a GC content of 39.09%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Lactobacillus rhamnosus has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)y\n",
      "Pediococcus pentosaceus has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)y\n",
      "Xanthomonas oryzae has a GC content of 63.7%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Bacillus has a GC content of 41.41%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Streptomyces has a GC content of 71.80%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 39.2%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Methylocystis has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Paenibacillus has a GC content of 46%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Enterococcus faecalis has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Enterobacter roggenkampii has a GC content of 56.16%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Unknown has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 39%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 0.9%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 27.6%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 53.8%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Unknown has a GC content of 64%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Methanococcus maripaludis has a GC content of 94%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Unknown has a GC content of 59.2%\n",
      "Is this GC content value correct? (y/n)n\n",
      "'Cellulomonas timonensis' has a GC content of 72.42%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Acidovorax citrulli has a GC content of 58.7%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Unknown has a GC content of 71%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 200%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Methylobacterium radiotolerans has a GC content of 70.5%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella has a GC content of 75%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Lactobacillus murinus has a GC content of 39.6%\n",
      "Is this GC content value correct? (y/n)y\n",
      "Unknown has a GC content of 68.66%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 95%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 34.4%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of Unknown\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 67.81%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 44.8%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 52.46%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Unknown has a GC content of 2%\n",
      "Is this GC content value correct? (y/n)n\n",
      "Here are the validated GC content values:\n",
      "E. coli has a GC content of 7.4%\n",
      "Oceanobacillus jordanicus has a GC content of 39.09%\n",
      "Paenibacillus has a GC content of 46%\n",
      "Enterobacter roggenkampii has a GC content of 56.16%\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30%\n",
      "Methanococcus maripaludis has a GC content of 94%\n",
      "'Cellulomonas timonensis' has a GC content of 72.42%\n",
      "Acidovorax citrulli has a GC content of 58.7%\n",
      "Methylobacterium radiotolerans has a GC content of 70.5%\n",
      "Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella has a GC content of 75%\n",
      "Lactobacillus murinus has a GC content of 39.6%\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import re\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "abstracts_with_one_perc = []\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Print the abstracts\n",
    "    noAbstract = 0\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            matches = re.findall(r\"\\%\", temp_record[\"Abstract\"][\"AbstractText\"][0])\n",
    "            if(len(matches) == 1):\n",
    "                abstracts_with_one_perc.append(temp_record)\n",
    "            pass\n",
    "        except KeyError:\n",
    "            print(\"No abstract available\")\n",
    "            noAbstract += 1\n",
    "    print(\"No abstract available for \" + str(noAbstract) + \" articles \"  + \"by total of \" + str(len(article_ids)) + \" articles\")\n",
    "\n",
    "hopeful_dicts = []\n",
    "\n",
    "for abstract in abstracts_with_one_perc:\n",
    "    text = abstract[\"Abstract\"][\"AbstractText\"][0]\n",
    "    bacterial_match = re.search(r\"<i>(.*?)</i>\", text)\n",
    "    if bacterial_match:\n",
    "        bacterial_name = bacterial_match.group(1)\n",
    "    else:\n",
    "        bacterial_name = \"Unknown\"\n",
    "    gc_match = re.search(r\"([\\d\\.]+\\%)\", text)\n",
    "    if gc_match:\n",
    "        gc_content = gc_match.group(1)\n",
    "    else:\n",
    "        gc_content = \"Unknown\"\n",
    "    print(f\"{bacterial_name} has a GC content of {gc_content}\")\n",
    "    is_correct = \"\"\n",
    "    while is_correct not in [\"y\", \"n\"]:\n",
    "        is_correct = input(\"Is this GC content value correct? (y/n)\")\n",
    "    if is_correct == \"y\":\n",
    "        is_valuable_data = \"yes\"\n",
    "    else:\n",
    "        is_valuable_data = \"no\"\n",
    "    hopefull_dict = {\"bacterial_name\": bacterial_name, \"gc_content\": gc_content, \"is_valuable_data\": is_valuable_data}\n",
    "    hopeful_dicts.append(hopefull_dict)\n",
    "\n",
    "# Show only dictionaries with a valuable key and value\n",
    "hopeful_dicts = [x for x in hopeful_dicts if x[\"bacterial_name\"] != \"Unknown\" and x[\"gc_content\"] != \"Unknown\" and x[\"is_valuable_data\"] == \"yes\"]\n",
    "\n",
    "print(\"Here are the validated GC content values:\")\n",
    "for d in hopeful_dicts:\n",
    "    print(f\"{d['bacterial_name']} has a GC content of {d['gc_content']}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "6cd40d0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Is the GC content value '80%' correct for Xanthomonas oryzae? (y/n) y\n",
      "Is the GC content value '96.9%' correct for Geminicoccaceae? (y/n) y\n",
      "Is the GC content value '5%' correct for Triticum aestivum? (y/n) n\n",
      "Is the GC content value '99.93%' correct for Petroselinum crispum? (y/n) y\n",
      "Is the GC content value '99.5%' correct for Brachybacterium muris? (y/n) n\n",
      "Is the GC content value '7.4%' correct for E. coli? (y/n) y\n",
      "Is the GC content value '96.2%' correct for Peptostreptococcus anaerobius? (y/n) n\n",
      "Is the GC content value '39.09%' correct for Oceanobacillus jordanicus? (y/n) y\n",
      "Is the GC content value '29.2%' correct for Acidobacteria? (y/n) y\n",
      "Is the GC content value '100%' correct for Thymus vulgaris? (y/n) y\n",
      "Is the GC content value '93.3%' correct for Oe? (y/n) y\n",
      "Is the GC content value '63.7%' correct for Xanthomonas oryzae? (y/n) y\n",
      "Is the GC content value '41.41%' correct for Bacillus? (y/n) y\n",
      "Is the GC content value '98.32%' correct for . Neobacillus massiliamazoniensis? (y/n) y\n",
      "Is the GC content value '68%' correct for Ralstonia solanacearum? (y/n) y\n",
      "Is the GC content value '94.73%' correct for Actinomycetales? (y/n) y\n",
      "Is the GC content value '46.6%' correct for Schleiferiaceae? (y/n) y\n",
      "Is the GC content value '71.80%' correct for Streptomyces? (y/n) y\n",
      "Is the GC content value '46%' correct for Paenibacillus? (y/n) y\n",
      "Is the GC content value '56.16%' correct for Enterobacter roggenkampii? (y/n) y\n",
      "Is the GC content value '36.49%' correct for Erysipelothrix rhusiopathiae? (y/n) y\n",
      "Is the GC content value '65.30%' correct for Pseudarthrobacter phenanthrenivorans? (y/n) y\n",
      "Is the GC content value '99.72%' correct for Allium fistulosum? (y/n) n\n",
      "Is the GC content value '5%' correct for n? (y/n) y\n",
      "Is the GC content value '57%' correct for Acidobacteria? (y/n) y\n",
      "Is the GC content value '35.23%' correct for Bacillus? (y/n) y\n",
      "Is the GC content value '93.2%' correct for Dielma fastidiosa? (y/n) y\n",
      "Is the GC content value '46.5%' correct for Bacillus velezensis? (y/n) y\n",
      "Is the GC content value '55%' correct for Agaricus bisporus.? (y/n) y\n",
      "Is the GC content value '53%' correct for Alteromonadales bacterium? (y/n) y\n",
      "Is the GC content value '94%' correct for Methanococcus maripaludis? (y/n) y\n",
      "Is the GC content value '89.81%' correct for Mauritia flexuosa? (y/n) y\n",
      "No abstract available\n",
      "Is the GC content value '62.9%' correct for Xanthomonas euvesicatoria? (y/n) y\n",
      "Is the GC content value '72.42%' correct for 'Cellulomonas timonensis'? (y/n) y\n",
      "Is the GC content value '58.7%' correct for Acidovorax citrulli? (y/n) y\n",
      "Is the GC content value '70.5%' correct for Methylobacterium radiotolerans? (y/n) y\n",
      "Is the GC content value '75%' correct for Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella? (y/n) n\n",
      "Is the GC content value '39.6%' correct for Lactobacillus murinus? (y/n) n\n",
      "Is the GC content value '92%' correct for Rubeoparvulum massiliense? (y/n) n\n",
      "Xanthomonas oryzae has a GC content of 80%\n",
      "Geminicoccaceae has a GC content of 96.9%\n",
      "Petroselinum crispum has a GC content of 99.93%\n",
      "E. coli has a GC content of 7.4%\n",
      "Oceanobacillus jordanicus has a GC content of 39.09%\n",
      "Acidobacteria has a GC content of 29.2%\n",
      "Thymus vulgaris has a GC content of 100%\n",
      "Oe has a GC content of 93.3%\n",
      "Xanthomonas oryzae has a GC content of 63.7%\n",
      "Bacillus has a GC content of 41.41%\n",
      ". Neobacillus massiliamazoniensis has a GC content of 98.32%\n",
      "Ralstonia solanacearum has a GC content of 68%\n",
      "Actinomycetales has a GC content of 94.73%\n",
      "Schleiferiaceae has a GC content of 46.6%\n",
      "Streptomyces has a GC content of 71.80%\n",
      "Paenibacillus has a GC content of 46%\n",
      "Enterobacter roggenkampii has a GC content of 56.16%\n",
      "Erysipelothrix rhusiopathiae has a GC content of 36.49%\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30%\n",
      "n has a GC content of 5%\n",
      "Acidobacteria has a GC content of 57%\n",
      "Bacillus has a GC content of 35.23%\n",
      "Dielma fastidiosa has a GC content of 93.2%\n",
      "Bacillus velezensis has a GC content of 46.5%\n",
      "Agaricus bisporus. has a GC content of 55%\n",
      "Alteromonadales bacterium has a GC content of 53%\n",
      "Methanococcus maripaludis has a GC content of 94%\n",
      "Mauritia flexuosa has a GC content of 89.81%\n",
      "Xanthomonas euvesicatoria has a GC content of 62.9%\n",
      "'Cellulomonas timonensis' has a GC content of 72.42%\n",
      "Acidovorax citrulli has a GC content of 58.7%\n",
      "Methylobacterium radiotolerans has a GC content of 70.5%\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import re\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "abstracts_with_gc = []\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Parse the abstracts\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            abstract = temp_record[\"Abstract\"][\"AbstractText\"][0]\n",
    "            gc_match = re.search(r\"([\\d\\.]+\\%)\", abstract)\n",
    "            bacterial_match = re.search(r\"<i>(.*?)</i>\", abstract)\n",
    "            if gc_match and bacterial_match:\n",
    "                gc_content = gc_match.group(1)\n",
    "                bacterial_name = bacterial_match.group(1)\n",
    "                # Prompt the user to input whether the GC content value is correct or not\n",
    "                correct = input(f\"Is the GC content value '{gc_content}' correct for {bacterial_name}? (y/n) \")\n",
    "                while correct.lower() not in ['y', 'n']:\n",
    "                    correct = input(\"Invalid input. Please enter 'y' or 'n': \")\n",
    "                if correct.lower() == 'y':\n",
    "                    abstracts_with_gc.append((bacterial_name, gc_content))\n",
    "        except KeyError:y\n",
    "            print(\"No abstract available\")\n",
    "\n",
    "# Print the bacterial names and their corresponding GC content percentages\n",
    "for bacterial_name, gc_content in abstracts_with_gc:\n",
    "    print(f\"{bacterial_name} has a GC content of {gc_content}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "c68deead",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Is the GC content value '80%' correct for Xanthomonas oryzae? (y/n) y\n",
      "Is the GC content value '96.9%' correct for Geminicoccaceae? (y/n) y\n",
      "Is the GC content value '5%' correct for Triticum aestivum? (y/n) y\n",
      "Is the GC content value '99.93%' correct for Petroselinum crispum? (y/n) y\n",
      "Is the GC content value '99.5%' correct for Brachybacterium muris? (y/n) y\n",
      "Is the GC content value '7.4%' correct for E. coli? (y/n) y\n",
      "Is the GC content value '96.2%' correct for Peptostreptococcus anaerobius? (y/n) y\n",
      "Is the GC content value '39.09%' correct for Oceanobacillus jordanicus? (y/n) y\n",
      "Is the GC content value '29.2%' correct for Acidobacteria? (y/n) y\n",
      "Is the GC content value '100%' correct for Thymus vulgaris? (y/n) y\n",
      "Is the GC content value '93.3%' correct for Oe? (y/n) y\n",
      "Is the GC content value '63.7%' correct for Xanthomonas oryzae? (y/n) y\n",
      "Is the GC content value '41.41%' correct for Bacillus? (y/n) y\n",
      "Is the GC content value '98.32%' correct for . Neobacillus massiliamazoniensis? (y/n) y\n",
      "Is the GC content value '68%' correct for Ralstonia solanacearum? (y/n) y\n",
      "Is the GC content value '94.73%' correct for Actinomycetales? (y/n) y\n",
      "Is the GC content value '46.6%' correct for Schleiferiaceae? (y/n) y\n",
      "Is the GC content value '71.80%' correct for Streptomyces? (y/n) y\n",
      "Is the GC content value '46%' correct for Paenibacillus? (y/n) y\n",
      "Is the GC content value '56.16%' correct for Enterobacter roggenkampii? (y/n) y\n",
      "Is the GC content value '36.49%' correct for Erysipelothrix rhusiopathiae? (y/n) y\n",
      "Is the GC content value '65.30%' correct for Pseudarthrobacter phenanthrenivorans? (y/n) y\n",
      "Is the GC content value '99.72%' correct for Allium fistulosum? (y/n) y\n",
      "Is the GC content value '5%' correct for n? (y/n) y\n",
      "Is the GC content value '57%' correct for Acidobacteria? (y/n) y\n",
      "Is the GC content value '35.23%' correct for Bacillus? (y/n) y\n",
      "Is the GC content value '93.2%' correct for Dielma fastidiosa? (y/n) y\n",
      "Is the GC content value '46.5%' correct for Bacillus velezensis? (y/n) y\n",
      "Is the GC content value '55%' correct for Agaricus bisporus.? (y/n) y\n",
      "Is the GC content value '53%' correct for Alteromonadales bacterium? (y/n) y\n",
      "Is the GC content value '94%' correct for Methanococcus maripaludis? (y/n) y\n",
      "Is the GC content value '89.81%' correct for Mauritia flexuosa? (y/n) y\n",
      "No abstract available\n",
      "Is the GC content value '62.9%' correct for Xanthomonas euvesicatoria? (y/n) y\n",
      "Is the GC content value '72.42%' correct for 'Cellulomonas timonensis'? (y/n) y\n",
      "Is the GC content value '58.7%' correct for Acidovorax citrulli? (y/n) y\n",
      "Is the GC content value '70.5%' correct for Methylobacterium radiotolerans? (y/n) y\n",
      "Is the GC content value '75%' correct for Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella? (y/n) y\n",
      "Is the GC content value '39.6%' correct for Lactobacillus murinus? (y/n) y\n",
      "Is the GC content value '92%' correct for Rubeoparvulum massiliense? (y/n) y\n",
      "Xanthomonas oryzae has a GC content of 80% (DOI: 37007514)\n",
      "Geminicoccaceae has a GC content of 96.9% (DOI: 36386637)\n",
      "Triticum aestivum has a GC content of 5% (DOI: 35698247)\n",
      "Petroselinum crispum has a GC content of 99.93% (DOI: 35695838)\n",
      "Brachybacterium muris has a GC content of 99.5% (DOI: 35392139)\n",
      "E. coli has a GC content of 7.4% (DOI: 35330182)\n",
      "Peptostreptococcus anaerobius has a GC content of 96.2% (DOI: 35313487)\n",
      "Oceanobacillus jordanicus has a GC content of 39.09% (DOI: 35095268)\n",
      "Acidobacteria has a GC content of 29.2% (DOI: 35087491)\n",
      "Thymus vulgaris has a GC content of 100% (DOI: 35041305)\n",
      "Oe has a GC content of 93.3% (DOI: 34739370)\n",
      "Xanthomonas oryzae has a GC content of 63.7% (DOI: 34598657)\n",
      "Bacillus has a GC content of 41.41% (DOI: 34311561)\n",
      ". Neobacillus massiliamazoniensis has a GC content of 98.32% (DOI: 34168882)\n",
      "Ralstonia solanacearum has a GC content of 68% (DOI: 33911932)\n",
      "Actinomycetales has a GC content of 94.73% (DOI: 33854785)\n",
      "Schleiferiaceae has a GC content of 46.6% (DOI: 33847553)\n",
      "Streptomyces has a GC content of 71.80% (DOI: 33707334)\n",
      "Paenibacillus has a GC content of 46% (DOI: 33033124)\n",
      "Enterobacter roggenkampii has a GC content of 56.16% (DOI: 32840429)\n",
      "Erysipelothrix rhusiopathiae has a GC content of 36.49% (DOI: 32512708)\n",
      "Pseudarthrobacter phenanthrenivorans has a GC content of 65.30% (DOI: 32284671)\n",
      "Allium fistulosum has a GC content of 99.72% (DOI: 32228807)\n",
      "n has a GC content of 5% (DOI: 31961776)\n",
      "Acidobacteria has a GC content of 57% (DOI: 31890783)\n",
      "Bacillus has a GC content of 35.23% (DOI: 31667287)\n",
      "Dielma fastidiosa has a GC content of 93.2% (DOI: 30949346)\n",
      "Bacillus velezensis has a GC content of 46.5% (DOI: 30944809)\n",
      "Agaricus bisporus. has a GC content of 55% (DOI: 30863388)\n",
      "Alteromonadales bacterium has a GC content of 53% (DOI: 30761333)\n",
      "Methanococcus maripaludis has a GC content of 94% (DOI: 30581420)\n",
      "Mauritia flexuosa has a GC content of 89.81% (DOI: 30544654)\n",
      "Xanthomonas euvesicatoria has a GC content of 62.9% (DOI: 30210484)\n",
      "'Cellulomonas timonensis' has a GC content of 72.42% (DOI: 29692905)\n",
      "Acidovorax citrulli has a GC content of 58.7% (DOI: 29422788)\n",
      "Methylobacterium radiotolerans has a GC content of 70.5% (DOI: 28982992)\n",
      "Enterococcus, Staphylococcus, Streptococcus, Serratia, Klebsiella has a GC content of 75% (DOI: 28459051)\n",
      "Lactobacillus murinus has a GC content of 39.6% (DOI: 28336601)\n",
      "Rubeoparvulum massiliense has a GC content of 92% (DOI: 28053703)\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "import re\n",
    "\n",
    "Entrez.email = \"emirhan.acikgoez@hotmail.com\"  # Always tell NCBI who you are\n",
    "\n",
    "# Search PubMed for articles with \"GC-content\" in the title\n",
    "abstracts_with_gc = []\n",
    "for i in range(0, 200, 20):\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=\"GC AND content AND bacterial[Title]\", retstart=i, retmax=20)\n",
    "    record = Entrez.read(handle)\n",
    "    ids = record[\"IdList\"]\n",
    "    # Process the IDs\n",
    "\n",
    "    # Get the list of article IDs\n",
    "    article_ids = record[\"IdList\"]\n",
    "\n",
    "    # Fetch the abstracts for the articles\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=article_ids, rettype=\"abstract\")\n",
    "    records = Entrez.read(handle)\n",
    "\n",
    "    # Parse the abstracts\n",
    "    for record in records[\"PubmedArticle\"]:\n",
    "        try:\n",
    "            temp_record = record[\"MedlineCitation\"][\"Article\"]\n",
    "            abstract = temp_record[\"Abstract\"][\"AbstractText\"][0]\n",
    "            gc_match = re.search(r\"([\\d\\.]+\\%)\", abstract)\n",
    "            bacterial_match = re.search(r\"<i>(.*?)</i>\", abstract)\n",
    "            doi = record['PubmedData']['ArticleIdList'][0] if 'ArticleIdList' in record['PubmedData'] else 'N/A'\n",
    "            if gc_match and bacterial_match:\n",
    "                gc_content = gc_match.group(1)\n",
    "                bacterial_name = bacterial_match.group(1)\n",
    "                # Prompt the user to input whether the GC content value is correct or not\n",
    "                correct = input(f\"Is the GC content value '{gc_content}' correct for {bacterial_name}? (y/n) \")\n",
    "                while correct.lower() not in ['y', 'n']:\n",
    "                    correct = input(\"Invalid input. Please enter 'y' or 'n': \")\n",
    "                if correct.lower() == 'y':\n",
    "                    abstracts_with_gc.append((bacterial_name, gc_content, doi))\n",
    "        except KeyError:\n",
    "            print(\"No abstract available\")\n",
    "\n",
    "# Print the bacterial names, their corresponding GC content percentages, and the DOI for each article\n",
    "for bacterial_name, gc_content, doi in abstracts_with_gc:\n",
    "    print(f\"{bacterial_name} has a GC content of {gc_content} (DOI: {doi})\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e53b1de5",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
